# Publication List

The BIH Cluster is a valuable resource.
It has been used to support the publications listed below.

- Please add your publications here.
- Acknowledge usage of the cluster in your manuscript as *"Computation has been performed on the HPC for Research cluster of the Berlin Institute of Health"*.

## Articles & Preprints

### Preprints

Merk T, Salehi S, Koehler RM, Cui Q, Olaru M, Hahn A, Provenza NR, Little S, Abbasi-Asl R, Starr PA, Neumann WJ (2025). “Pre-trained Transformer-models using chronic invasive electrophysiology for symptom decoding without patient-individual training.” arXiv preprint arXiv:2508.10160. [☞ Link](https://arxiv.org/abs/2508.10160)

Chopra A, Song J, Weiner J, Beule D, Schaefer AS (2024). “Genetic analysis of cis-enhancers associated with bone mineral density and periodontitis in
the gene SOST.” _ResearchSquare_.  [☞ Link](https://www.researchsquare.com/article/rs-4409923/v1)

Opitz B, Essex M, Pascual-Leone BM, Loeber U, Kuhring M, Zhang B, Bruening U, Fritsche-Guenther R, Krzanowski M, Vernengo FF, Brumhard S, Röwekamp I,
Bielecka A, Lesker T, Wyler E, Landthaler M, Mantei A, Meisel C, Caesar S, Thibeault C, Corman V, Marko L, Suttorp N, Strowig T, Kurth F, Sander L, Li
Y, Kirwan J, Forslund S (2023). “Gut microbiota dysbiosis is associated with altered tryptophan metabolism and dysregulated inflammatory response in
severe COVID-19.” doi:10.21203/rs.3.rs-2459463/v1  [☞ Link](https://www.researchsquare.com/article/rs-2459463/v1)

Zuljan E, von der Emde B, Piwonski I, Pestana ACA, Klinghammer K, Mock A, Horak P, Heining C, Klauschen F, Pretzell M, Boerries I, Brandts C,
Kreutzfeldt S, Teleanu M, Hübschmann D, Morris L, Keller U, Glimm H, Fröhling S, Ochsenreither S, Keilholz U, Blanc E, Beule D, Rieke D (2024).
“Integrative analysis reveals a macrophage-predominant, immunosuppressive immune microenvironment and subtype-specific therapeutic vulnerabilities in
advanced salivary gland cancer.” _medRxiv_, 2024-06. doi:10.1101/2024.06.11.24308538  [☞ Link](https://www.medrxiv.org/content/10.1101/2024.06.11.24308538v2)

Yerinde C, Keye J, Durlanik S, Freise I, Nowak F, Hsiao H, Letizia M, Schlickeiser S, Obermayer B, Huck A, Friedrich M, Wu H, Kunkel D, Kühl AA, Bauer
S, Thiel A, Siegmund B, Glauben R, Weidinger C (2022). “HDAC7 controls anti-viral and anti-tumor immunity by CD8+ T cells.” _bioRxiv_.
doi:10.1101/2022.09.18.508452  [☞ Link](https://www.biorxiv.org/content/early/2022/09/19/2022.09.18.508452)

Winkler W, Díaz C, Blanc E, Napieczynska H (2021). “Preclinical models of human multiple myeloma subgroups.” _bioRxiv_. doi:10.1101/2021.08.28.458010
 [☞ Link](https://www.biorxiv.org/content/10.1101/2021.08.28.458010.abstract)

Göppel T, Obermayer B, Chen I, Gerland U (2021). “A kinetic error filtering mechanism for enzyme-free copying of nucleic acid sequences.” _bioRxiv_.
doi:10.1101/2021.08.06.455386  [☞ Link](https://www.biorxiv.org/content/10.1101/2021.08.06.455386.abstract)

### 2025

Merk T, Köhler RM, Brotons TM, Vossberg SR, Peterson V, Lyra LF, Vanhoecke J, Chikermane M, Binns TS, Li N, Walton A, Neudorfer C, Bush A, Sisterson N, Busch J, Lofredi R, Habets J, Huebl J, Zhu G, Yin Z, Zhao B, Merkl A, Bajbouj M, Krause P, Faust K, Schneider GH, Horn A, Zhang J, Kühn AA, Richardson RM, Neumann WJ (2025). “Invasive neurophysiology and whole brain connectomics for neural decoding in patients with brain implants.” Nature Biomedical Engineering, 1–18. Nature Publishing Group. [doi: 10.1038/s41551-025-01467-9](https://doi.org/10.1038/s41551-025-01467-9)

Sebastian Röner, Lea Burkard, Michael R Speicher, Martin Kircher, cfDNA UniFlow: a unified preprocessing pipeline for cell-free DNA data from liquid biopsies, GigaScience, Volume 13, 2024, giae102, [doi: 10.1093/gigascience/giae102](https://doi.org/10.1093/gigascience/giae102)

P. Keukeleire, J.D. Rosen, A. Göbel-Knapp, K. Salomon, M. Schubach, M. Kircher. Using individual barcodes to increase quantification power of massively parallel reporter assays. *BMC Bioinformatics* 2025 26, 52. [doi: 10.1186/s12859-025-06065-9](https://doi.org/10.1186/s12859-025-06065-9)

### 2024

Max Schubach, Thorben Maass, Lusiné Nazaretyan, Sebastian Röner, Martin Kircher, CADD v1.7: using protein language models, regulatory CNNs and other nucleotide-level scores to improve genome-wide variant predictions, Nucleic Acids Research, Volume 52, Issue D1, 5 January 2024, Pages D1143–D1154, [doi: 10.1093/nar/gkad989](https://doi.org/10.1093/nar/gkad989)

Köhler RM, Binns TS, Merk T, Zhu G, Yin Z, Zhao B, Chikermane M, Vanhoecke J, Busch JL, Habets JGV, Faust K, Schneider GH, Cavallo A, Haufe S, Zhang J, Kühn AA, Haynes JD, Neumann WJ. Dopamine and deep brain stimulation accelerate the neural dynamics of volitional action in Parkinson’s disease. *Brain*. 2024 Oct 1;147(10):3358–69. [doi: 10.1093/brain/awae219](https://doi.org/10.1093/brain/awae219)

Binns TS, Köhler RM, Vanhoecke J, Chikermane M, Gerster M, Merk T, Pellegrini F, Busch JL, Habets JGV, Cavallo A, Beyer JC, Al-Fatly B, Li N, Horn A, Krause P, Faust K, Schneider GH, Haufe S, Kühn AA, Neumann WJ. Shared pathway-specific network mechanisms of dopamine and deep brain stimulation for the treatment of Parkinson’s disease. *bioRxiv*. 2024; p. 2024.04.14.586969. Available from: [doi: 10.1101/2024.04.14.586969v2](https://www.biorxiv.org/content/10.1101/2024.04.14.586969v2)

Hollunder, B., Ostrem, J.L., Sahin, I.A., Rajamani, N., Oxenford, S., Butenko, K., Neudorfer, C., Reinhardt, P., Zvarova, P., Polosan, M., Akram, H., Vissani, M., Zhang, C., Sun, B., Navratil, P., Reich, M.M., Volkmann, J., Yeh, F.-C., Baldermann, J.C., Dembek, T.A., Visser-Vandewalle, V., Alho, E.J.L., Franceschini, P.R., Nanda, P., Finke, C., Kühn, A.A., Dougherty, D.D., Richardson, R.M., Bergman, H., DeLong, M.R., Mazzoni, A., Romito, L.M., Tyagi, H., Zrinzo, L., Joyce, E.M., Chabardes, S., Starr, P.A., Li, N., Horn, A., 2024. Mapping dysfunctional circuits in the frontal cortex using deep brain stimulation. Nat. Neurosci. 1–14. [doi: 10.1038/s41593-024-01570-1](https://doi.org/10.1038/s41593-024-01570-1)


Schmidt A, Danyel M, Grundmann K, Brunet T, Klinkhammer H, Hsieh T, Engels H, Peters S, Knaus A, Moosa S, Averdunk L, Boschann F, Sczakiel HL,
Schwartzmann S, Mensah MA, Pantel JT, Holtgrewe M, Bösch A, Weiß C, Weinhold N, Suter A, Stoltenburg C, Neugebauer J, Kallinich T, Kaindl AM, Holzhauer
S, Bührer C, Bufler P, Kornak U, Ott C, Schülke M, Nguyen HHP, Hoffjan S, Grasemann C, Rothoeft T, Brinkmann F, Matar N, Sivalingam S, Perne C, Mangold
E, Kreiss M, Cremer K, Betz RC, Mücke M, Grigull L, Klockgether T, Spier I, Heimbach A, Bender T, Brand F, Stieber C, Morawiec AM, Karakostas P, Schäfer
VS, Bernsen S, Weydt P, Castro-Gomez S, Aziz A, Grobe-Einsler M, Kimmich O, Kobeleva X, Önder D, Lesmann H, Kumar S, Tacik P, Basin MA, Incardona P,
Lee-Kirsch MA, Berner R, Schuetz C, Körholz J, Kretschmer T, Donato ND, Schröck E, Heinen A, Reuner U, Hanßke A, Kaiser FJ, Manka E, Munteanu M,
Kuechler A, Cordula K, Hirtz R, Schlapakow E, Schlein C, Lisfeld J, Kubisch C, Herget T, Hempel M, Weiler-Normann C, Ullrich K, Schramm C, Rudolph C,
Rillig F, Groffmann M, Muntau A, Tibelius A, Schwaibold EMC, Schaaf CP, Zawada M, Kaufmann L, Hinderhofer K, Okun PM, Kotzaeridou U, Hoffmann GF,
Choukair D, Bettendorf M, Spielmann M, Ripke A, Pauly M, Münchau A, Lohmann K, Hüning I, Hanker B, Bäumer T, Herzog R, Hellenbroich Y, Westphal DS,
Strom T, Kovacs R, Riedhammer KM, Mayerhanser K, Graf E, Brugger M, Hoefele J, Oexle K, Mirza-Schreiber N, Berutti R, Schatz U, Krenn M, Makowski C,
Weigand H, Schröder S, Rohlfs M, Vill K, Hauck F, Borggraefe I, Müller-Felber W, Kurth I, Elbracht M, Knopp C, Begemann M, Kraft F, Lemke JR, Hentschel
J, Platzer K, Strehlow V, Jamra RA, Kehrer M, Demidov G, Beck-Wödl S, Graessner H, Sturm M, Zeltner L, Schöls LJ, Magg J, Bevot A, Kehrer C, Kaiser N,
Turro E, Horn D, Grüters-Kieslich A, Klein C, Mundlos S, Nöthen M, Riess O, Meitinger T, Krude H, Krawitz PM, Haack T, Ehmke N, Wagner M (2024).
“Next-generation phenotyping integrated in a national framework for patients with ultrarare disorders improves genetic diagnostics and yields new
molecular findings.” _Nature Genetics_. ISSN 1546-1718, doi:10.1038/s41588-024-01836-1  [☞ Link](https://doi.org/10.1038/s41588-024-01836-1)

Marot-Lassauzaie V, Beneyto-Calabuig S, Obermayer B, Velten L, Beule D, Haghverdi L (2024). “Identifying cancer cells from calling single-nucleotide
variants in scRNA-seq data.” _Bioinformatics_, 40(9), btae512. ISSN 1367-4811, doi:10.1093/bioinformatics/btae512
 [☞ Link](https://doi.org/10.1093/bioinformatics/btae512)

Horn V, Cancino CA, Steinheuer LM, Obermayer B, Fritz K, Nguyen AL, Juhran KS, Plattner C, Bösel D, Oldenburg L, Burns M, Schulz AR, Saliutina M,
Mantzivi E, Lissner D, Conrad T, Mashreghi M, Zundler S, Sonnenberg E, Schumann M, Haag L, Beule D, Flatz L, Atreya I, Atreya R, Bacher P, Becker C,
Bojarski C, Britzen-Laurent N, Bosch-Voskens C, Chang H, Diefenbach A, Günther C, Hegazy AN, Hildner K, Klose CS, Koop K, Krug S, Kühl AA, Leppkes M,
López-Posadas R, Ludwig LS, Neufert C, Neurath M, Patankar J, Prüß M, Radbruch A, Romagnani C, Ronchi F, Sanders A, Scheffold A, Schulzke J, Schumann M,
Schürmann S, Siegmund B, Stürzl M, Trajanoski Z, Triantafyllopoulou A, Waldner M, Weidinger C, Wirtz S, Zundler S, Trjanoski Z, D’Haens G, Weidinger C,
Mei HE, Siegmund B, Thurley K, Hegazy AN (2024). “Multimodal Profiling of Peripheral Blood Identifies Proliferating Circulating Effector CD4+ T Cells as
Predictors for Response to Integrin α4β7–Blocking Therapy in Inflammatory Bowel Disease.” _Gastroenterology_. ISSN 0016-5085,
doi:10.1053/j.gastro.2024.09.021  [☞ Link](https://www.sciencedirect.com/science/article/pii/S0016508524055276)

Menzel M, Martis-Thiele M, Goldschmid H, Ott A, Romanovsky E, Siemanowski-Hrach J, Seillier L, Brüchle NO, Maurer A, Lehmann K, Begemann M, Elbracht M,
Meyer R, Dintner S, Claus R, Meier-Kolthoff JP, Blanc E, Möbs M, Joosten M, Benary M, Basitta P, Hölscher F, Tischler V, Groß T, Kutz O, Prause R,
William D, Horny K, Goering W, Sivalingam S, Borkhardt A, Blank C, Junk SV, Yasin L, Moskalev EA, Carta MG, Ferrazzi F, Tögel L, Wolter S, Adam E,
Matysiak U, Rosenthal T, Dönitz J, Lehmann U, Schmidt G, Bartels S, Hofmann W, Hirsch S, Dikow N, Göbel K, Banan R, Hamelmann S, Fink A, Ball M, Neumann
O, Rehker J, Kloth M, Murtagh J, Hartmann N, Jurmeister P, Mock A, Kumbrink J, Jung A, Mayr E, Jacob A, Trautmann M, Kirmse S, Falkenberg K, Ruckert C,
Hirsch D, Immel A, Dietmaier W, Haack T, Marienfeld R, Fürstberger A, Niewöhner J, Gerstenmaier U, Eberhardt T, Greif PA, Appenzeller S, Maurus K, Doll
J, Jelting Y, Jonigk D, Märkl B, Beule D, Horst D, Wulf A, Aust D, Werner M, Reuter-Jessen K, Ströbel P, Auber B, Sahm F, Merkelbach-Bruse S, Siebolts
U, Roth W, Lassmann S, Klauschen F, Gaisa NT, Weichert W, Evert M, Armeanu-Ebinger S, Ossowski S, Schroeder C, Schaaf CP, Malek N, Schirmacher P, Kazdal
D, Pfarr N, Budczies J, Stenzinger A (2024). “Benchmarking whole exome sequencing in the German network for personalized medicine.” _European Journal of
Cancer_, 211, 114306. ISSN 0959-8049, doi:10.1016/j.ejca.2024.114306  [☞ Link](https://www.sciencedirect.com/science/article/pii/S0959804924009626)

Fatykhova D, Fritsch VN, Siebert K, Methling K, Lalk M, Busche T, Kalinowski J, Weiner J, Beule D, Bertrams W, others (2024). “Microenvironmental
acidification by pneumococcal sugar consumption fosters barrier disruption and immune suppression in the human alveolus.” _European Respiratory
Journal_. doi:doi.org/10.1183/13993003.01983-2023  [☞ Link](https://publications.ersnet.org/content/erj/early/2024/08/22/1399300301983-2023)

Adhikary PP, Idowu T, Tan Z, Hoang C, Shanta S, Dumbani M, Mappalakayil L, Awasthi B, Bermudez M, Weiner J, Beule D, Wolber G, Page BD, Hedtrich S
(2024). “Disrupting TSLP-TSLP receptor interactions via putative small molecule inhibitors yields a novel and efficient treatment option for atopic
diseases.” _EMBO Molecular Medicine_, 1-27. doi:10.1038/s44321-024-00085-3  [☞ Link](https://www.embopress.org/doi/full/10.1038/s44321-024-00085-3)

Mattei D, Ivanov A, Hammer J, Ugursu B, Schalbetter S, Richetto J, Weber-Stadlbauer U, Mueller F, Scarborough J, Wolf SA, Kettenmann H, Wollscheid B,
Beule D, Meyer U (2024). “Microglia undergo molecular and functional adaptations to dark and light phases in male laboratory mice.” _Brain, Behavior,
and Immunity_, 120, 571-583. ISSN 0889-1591, doi:10.1016/j.bbi.2024.07.007  [☞ Link](https://www.sciencedirect.com/science/article/pii/S088915912400477X)

Kopp J, Koch LA, Lyubenova H, Küchler O, Holtgrewe M, Ivanov A, Dubourg C, Launay E, Brachs S, Mundlos S, Pellerin L, Mameli M, Paolicelli RC (2024).
“Loss-of-function variants affecting the STAGA complex component SUPT7L cause a developmental disorder with generalized lipodystrophy.” _Human
Genetics_, 143(5), 683-694. doi:10.1007/s00439-024-02669-y  [☞ Link](https://link.springer.com/article/10.1007/s00439-024-02669-y)

Ruiz-Orera J, Miller DC, Greiner J, Genehr C, Grammatikaki A, Blachut S, Mbebi J, Patone G, Myronova A, Adami E, others (2024). “Evolution of
translational control and the emergence of genes and open reading frames in human and non-human primate hearts.” _Nature Cardiovascular Research_, 1-19.
doi:10.1038/s44161-024-00544-7  [☞ Link](https://www.nature.com/articles/s44161-024-00544-7)

Fisch A, Pestana A, Sachse V, Doll C, Hofmann E, Heiland M, Obermueller T, Heidemann J, Dommerich S, Schoppe D, Schallenberg S, Piwonski I, Blanc E,
Tinhofer I (2024). “Feasibility analysis of using patient-derived tumour organoids for treatment decision guidance in locally advanced head and neck
squamous cell carcinoma.” _European journal of cancer (Oxford, England : 1990)_, 213, 115100. ISSN 1879-0852, doi:10.1016/j.ejca.2024.115100
 [☞ Link](https://linkinghub.elsevier.com/retrieve/pii/S0959804924017076)

Al-Diab O, Sünkel C, Blanc E, Catar RA, Ashraf MI, Zhao H, Wang P, Rinschen MM, Fritsche-Guenther R, Grahammer F, Bachmann S, Beule D, Kirwan JA,
Rajewsky N, Huber TB, Gürgen D, Kusch A (2024). “Sex-specific molecular signature of mouse podocytes in homeostasis and in response to pharmacological
challenge with rapamycin.” _Biology of sex differences_, 15, 72. ISSN 2042-6410, doi:10.1186/s13293-024-00647-7
 [☞ Link](https://bsd.biomedcentral.com/articles/10.1186/s13293-024-00647-7)

Trinks A, Milek M, Beule D, Kluge J, Florian S, Sers C, Horst D, Morkel M, Bischoff P (2024). “Robust detection of clinically relevant features in
single-cell RNA profiles of patient-matched fresh and formalin-fixed paraffin-embedded (FFPE) lung cancer tissue.” _Cellular Oncology_, 47, 1-11.
doi:10.5281/zenodo.7852154  [☞ Link](https://link.springer.com/article/10.1007/s13402-024-00922-0)

Šmid A, Štajdohar M, Milek M, Urbančič D, Kuželički NK, Tamm R, Metspalu A, Mlinarič-Raščan I (2024). “Transcriptome analysis reveals involvement of
thiopurine S-methyltransferase in oxidation-reduction processes.” _European Journal of Pharmaceutical Sciences_, 192, 106616.
doi:10.1016/j.ejps.2023.106616  [☞ Link](https://www.sciencedirect.com/science/article/pii/S0928098723002464)

Dörner PJ, Anandakumar H, Röwekamp I, Vernengo FF, Pascual-Leone BM, Krzanowski M, Sellmaier J, Brüning U, Fritsche-Guenther R, Pfannkuch L, Kurth F,
Milek M, Igbokwe V, Löber U, Gutbier B, Holstein M, Heinz GA, Mashreghi M, Schulte LN, Klatt A, Caesar S, Wienhold S, Offermanns S, Mack M, Witzenrath
M, Jordan S, Beule D, Kirwan JA, Forslund SK, Wilck N, Bartolomaeus H, Heimesaat MM, Opitz B (2024). “Clinically used broad-spectrum antibiotics
compromise inflammatory monocyte-dependent antibacterial defense in the lung.” _Nature Communications_, 15(1), 2788.

Röwekamp I, Maschirow L, Rabes A, Vernengo FF, Hamann L, Heinz GA, Mashreghi M, Caesar S, Milek M, Fonseca ACF, Wienhold S, Nouailles G, Yao L, Mousavi
S, Bruder D, Boehme JD, Puzianowska-Kuznicka M, Beule D, Witzenrath M, Group CS, Löhning M, Klose CSN, Heimesaat MM, Diefenbach A, Opitz B, Fuchs A,
Engelmann M, Paul G, Ayoub M, Groehl K, Riedl K, Stolz D, Bauer W, Diehl-Wiesenecker EC, Terue IvW, Galtung N, Suttorp N, Witzenrath M, Wildberg C, Pley
C, Zessin E, Schmager S, Schaaf B, Kremling J, Nickoleit-Bitzenberger D, Azzaui H, Hower M, Hempel F, Prebeg K, Popkirova K, Kolditz M, Schulte-Hubbert
B, Langner S, Rohde G, Bellinghausen C, Grϋnewaldt A, Endres A, Frigerio C, Fiedler B, Panning M, Welte T, Pink I, Drick N, Fϋhner T, Klooster Mv,
Steinberg T, Barten-Neiner G, Kröner W, Unruh O, Adaskina N, Eberhardt F, Julius C, Illig T, Klopp N, Pletz M, Schleenvoigt BT, Bahrs C, Moeser A,
Ankert J, Sommerwerck U, Wintermantel T, Drömann D, Parschke P, Franzen K, Rupp J, Waldeck F, Käding N, Spinner C, Erber J, Voit F, Schneider J, Falcone
M, Tiseo G, Heigener D, Hering I, Albrich W, Rassouli F, Wirth B, Neurohr C, Essig A, Stenger S, Wallner M, Burgmann H, Traby L, Schubert L, nurses aas
(2024). “IL-33 controls IL-22-dependent antibacterial defense by modulating the microbiota.” _Proceedings of the National Academy of Sciences_,
121(22), e2310864121. doi:10.1073/pnas.2310864121  [☞ Link](https://www.pnas.org/doi/abs/10.1073/pnas.2310864121)

Wei T, Blanc E, Peidli S, Bischoff P, Trinks A, Horst D, Sers C, Blüthgen N, Beule D, Morkel M, Obermayer B (2024). “High-confidence calling of normal
epithelial cells allows identification of a novel stem-like cell state in the colorectal cancer microenvironment.” _International Journal of Cancer_,
n/a(n/a). doi:10.1002/ijc.35079  [☞ Link](https://onlinelibrary.wiley.com/doi/abs/10.1002/ijc.35079)

Hramyka D, Sczakiel HL, Zhao MX, Stolpe O, Nieminen M, Adam R, Danyel M, Einicke L, Hägerling R, Knaus A, Mundlos S, Schwartzmann S, Seelow D, Ehmke N,
Mensah MA, Boschann F, Beule D, Holtgrewe M (2024). “REEV: review, evaluate and explain variants.” _Nucleic Acids Research_, gkae366.
doi:10.1093/nar/gkae366  [☞ Link](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkae366/7676836)

Ramberger E, Sapozhnikova V, Ng YLD, Dolnik A, Ziehm M, Popp O, Str"ang E, Kull M, Gr"unschl"ager F, Kr"uger J, Benary M, M"uller S, Gao X, Murgai A,
Haji M, Schmidt A, Lutz R, Nogai A, Braune J, Laue D, Langer C, Khandanpour C, Bassermann F, D"ohner H, Engelhardt M, Straka C, Hundemer M, Beule D,
Haas S, Keller U, Einsele H, Bullinger L, Knop S, Mertins P, Kr"onke J (2024). “The proteogenomic landscape of multiple myeloma reveals insights into
disease biology and therapeutic opportunities.” _Nature Cancer_. ISSN 2662-1347, doi:10.1038/s43018-024-00784-3
 [☞ Link](https://doi.org/10.1038/s43018-024-00784-3)

Habringer S, Ihlow J, Kleo K, Klostermann A, Schmidt M, Chai L, Knödler M, Leyvraz S, Sigler C, Sinn B, Maschmeyer G, Jegodka Y, Benary M, Ott C,
Tinhofer I, Schäfer R, Möbs M, Keller U, Keilholz U, Rieke DT (2024). “A diagnostic challenge of KIT p.V559D and BRAF p.G469A mutations in a paragastric
mass.” _The Oncologist_, oyae137. ISSN 1083-7159, doi:10.1093/oncolo/oyae137  [☞ Link](https://doi.org/10.1093/oncolo/oyae137)

Shi W, Song J, Weiner J, Chopra A, Dommisch H, Beule D, Schaefer AS (2024). “lncRNA CDKN2B-AS1 regulates collagen expression.” _Human Genetics_.
doi:10.1007/s00439-024-02674-1  [☞ Link](https://doi.org/10.1007/s00439-024-02674-1)

Shi W, Song J, Dommisch H, Weiner J, Schaefer A (2024). “STAT1 binds to CAD and periodontitis risk SNP rs10757278, causing inhibition of lncRNA
CDKN2B-AS1 expression, which activates transcription of collagen genes in cells of the extracellular matrix.” In _EUROPEAN JOURNAL OF HUMAN GENETICS_,
volume 32, 402-402. SPRINGERNATURE CAMPUS, 4 CRINAN ST, LONDON, N1 9XW, ENGLAND.

Zheng L, Chopra A, Weiner J, Dommisch H, Schaefer A (2024). “Systematic search for microRNA regulated genes with roles in the etiology of
periodontitis.” In _EUROPEAN JOURNAL OF HUMAN GENETICS_, volume 32, 406-406. SPRINGERNATURE CAMPUS, 4 CRINAN ST, LONDON, N1 9XW, ENGLAND.

Lebedin M, Ratswohl C, Garg A, Schips M, García CV, Spatt L, Thibeault C, Obermayer B, Weiner J, Velásquez IM, others (2024). “Soluble ACE2 correlates
with severe COVID-19 and can impair antibody responses.” _iScience_.

Tranter E, Frentsch M, Hütter-Krönke ML, Vuong GL, Busch D, Loyal L, Henze L, Rosnev S, Blau I, Thiel A, Beule D, Bullinger L, Obermayer B, Na I (2024).
“Comparable CD8+ T-cell responses to SARS-CoV-2 vaccination in single-cell transcriptomics of recently allogeneic transplanted patients and healthy
individuals.” _Journal of Medical Virology_, 96(3), e29539. doi:10.1002/jmv.29539  [☞ Link](https://onlinelibrary.wiley.com/doi/abs/10.1002/jmv.29539)

Bhasin MA, Knaus A, Incardona P, Schmid A, Holtgrewe M, Elbracht M, Krawitz PM, Hsieh T (2024). “Enhancing Variant Prioritization in VarFish through
On-Premise Computational Facial Analysis.” _Genes_, 15(3), 370.

Ermshaus A, Piechotta M, Rüter G, Keilholz U, Leser U, Benary M (2024). “preon: Fast and accurate entity normalization for drug names and cancer types
in precision oncology.” _Bioinformatics_, 40(3), btae085. ISSN 1367-4811, doi:10.1093/bioinformatics/btae085
 [☞ Link](https://doi.org/10.1093/bioinformatics/btae085)

Jahn D, Knapstein PR, Otto E, Köhli P, Sevecke J, Graef F, Graffmann C, Fuchs M, Jiang S, Rickert M, Erdmann C, Appelt J, Revend L, Küttner Q, Witte J,
Rahmani A, Duda G, Xie W, Donat A, Schinke T, Ivanov A, Tchouto MN, Beule D, Frosch K, Baranowsky A, Tsitsilonis S, Keller J (2024). “Increased
beta2-adrenergic signaling promotes fracture healing through callus neovascularization in mice.” _Science Translational Medicine_, 16(743), eadk9129.
doi:10.1126/scitranslmed.adk9129  [☞ Link](https://www.science.org/doi/10.1126/scitranslmed.adk9129)


### 2023

Max Schubach, Lusiné Nazaretyan, Martin Kircher, The Regulatory Mendelian Mutation score for GRCh38, GigaScience, Volume 12, 2023, giad024, [doi: 10.1093/gigascience/giad024](https://doi.org/10.1093/gigascience/giad024)

Monsorno K, Ginggen K, Ivanov A, Buckinx A, Lalive AL, Tchenio A, Benson S, Vendrell M, D’Alessandro A, Beule D, others (2023). “Loss of microglial MCT4
leads to defective synaptic pruning and anxiety-like behavior in mice.” _Nature Communications_, 14(1), 5749. doi:10.1038/s41467-023-41502-4
 [☞ Link](https://www.nature.com/articles/s41467-023-41502-4)

Markus D, Pelletier A, Boube M, Port F, Boutros M, Payre F, Obermayer B, Zanet J (2023). “The pleiotropic functions of Pri smORF peptides synchronize
leg development regulators.” _PLOS Genetics_, 19(10), 1-27. doi:10.1371/journal.pgen.1011004  [☞ Link](https://doi.org/10.1371/journal.pgen.1011004)

Ludwik KA, Opitz R, Jyrch S, Megges M, Weiner J, Beule D, Kühnen P, Stachelscheid H (2023). “Generation of iPSC lines with SLC16A2: G410R or SLC16A2
knock out.” _Stem Cell Research_, 103256. doi:10.1016/j.scr.2023.103256  [☞ Link](https://www.sciencedirect.com/science/article/pii/S1873506123002428)

Schneemann M, Heils L, Moos V, Weiß F, Krug SM, Weiner J, Beule D, Gerhard R, Schulzke J, Bücker R (2023). “A Colonic Organoid Model Challenged with the
Large Toxins of Clostridioides difficile TcdA and TcdB Exhibit Deregulated Tight Junction Proteins.” _Toxins_, 15(11), 643. doi:10.3390/toxins15110643
 [☞ Link](https://www.mdpi.com/2072-6651/15/11/643)

Monsorno K, Ginggen K, Ivanov A, Buckinx A, Lalive AL, Tchenio A, Benson S, Vendrell M, D’Alessandro A, Beule D, Pellerin L, Mameli M, Paolicelli RC
(2023). “Loss of microglial MCT4 leads to defective synaptic pruning and anxiety-like behavior in mice.” _Nature Communications_, 14(1), 5749.

Zheng L, Chopra A, Weiner J, Beule D, Dommisch H, Schaefer A (2023). “miRNAs from inflamed gingiva link gene signaling to increased MET expression.”
_Journal of Dental Research_, 00220345231197984. doi:10.1177/002203452311979  [☞ Link](https://journals.sagepub.com/doi/full/10.1177/00220345231197984)

Mhlekude B, Postmus D, Stenzel S, Weiner J, Jansen J, Zapatero-Belinchòn FJ, Olmer R, Richter A, Heinze J, Heinemann N, Mühlemann B, Schroeder S, Jones
TC, Müller MA, Drosten C, Pich A, Thiel V, Martin U, Niemeyer D, Gerold G, Beule D, Goffinet C (2023). “Pharmacological inhibition of bromodomain and
extra-terminal proteins induces an NRF-2-mediated antiviral state that is subverted by SARS-CoV-2 infection.” _PLoS pathogens_, 19(9), e1011657.
doi:10.1371/journal.ppat.1011657  [☞ Link](https://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1011657)

Benary M, Wang XD, Schmidt M, Soll D, Hilfenhaus G, Nassir M, Sigler C, Knödler M, Keller U, Beule D, Keilholz U, Leser U, Rieke DT (2023). “Leveraging
Large Language Models for Decision Support in Personalized Oncology.” _JAMA Network Open_, 6(11), e2343689-e2343689.
doi:10.1001/jamanetworkopen.2023.43689 

Briest F, Noerenberg D, Hennch C, Yoshida K, Hablesreiter R, Nimo J, Sasca D, Kirchner M, Mansouri L, Inoue Y, Wiegand L, Staiger AM, Casadei B,
Korkolopoulou P, Weiner J, Lopez-Guillermo A, Warth A, Schneider T, Nagy Á, Klapper W, Hummel M, Kanellis G, Anagnostopoulos I, Mertins P, Bullinger L,
Rosenquist R, Vassilakopoulos TP, Ott G, Ogawa S, Damm F (2023). “Frequent ZNF217 mutations lead to transcriptional deregulation of interferon signal
transduction via altered chromatin accessibility in B cell lymphoma.” _Leukemia_, 1-13. doi:10.1038/s41375-023-02013-9
 [☞ Link](https://www.nature.com/articles/s41375-023-02013-9)

Spranger L, Weiner J, Bredow J, Zeitz U, Grittner U, Boschmann M, Dickmann S, Stobäus N, Schwartzenberg RJ, Brachs M, Spranger J, Mai K (2023). “Thrifty
energy phenotype predicts weight regain in postmenopausal women with overweight or obesity and is related to FGFR1 signaling.” _Clinical Nutrition_,
42(4), 559-567. doi:10.1016/j.clnu.2023.02.020  [☞ Link](https://www.sciencedirect.com/science/article/abs/pii/S0261561423000560)

Chopra A, Song J, Weiner J, Keceli HG, Dincer PR, Cruz R, Carracedo A, Blanco J, Dommisch H, Schaefer AS (2023). “RSPO4 is a potential risk gene of
stages III-IV, grade C periodontitis through effects on innate immune response and oral barrier integrity.” _Journal of Clinical Periodontology_,
50(4), 476-486. doi:10.1111/jcpe.13758  [☞ Link](https://onlinelibrary.wiley.com/doi/full/10.1111/jcpe.13758)

Lechner L, Opitz R, Silver MJ, Krabusch PM, Prentice AM, Field MS, Stachelscheid H, Leitão E, Schröder C, Vallone VF, Horsthemke B, Jöckel K, Schmidt B,
Nöthen MM, Hoffmann P, Herms S, Kleyn PW, Megges M, Blume-Peytavi U, Weiss K, Mai K, Blankenstein O, Obermayer B, Wiegand S, Kühnen PKP (2023).
“Early-set POMC methylation variability is accompanied by increased risk for obesity and is addressable by MC4R agonist treatment.” _Science
Translational Medicine_, 15(705), eadg1659. doi:10.1126/scitranslmed.adg1659  [☞ Link](https://www.science.org/doi/abs/10.1126/scitranslmed.adg1659)

Obermayer B, Keilholz L, Conrad T, Frentsch M, Blau I, Vuong L, Lesch S, Movasshagi K, Tietze-Stolley C, Loyal L, Henze L, Penack O, Stervbo U, Babel N,
Haas S, Beule D, Bullinger L, Wittenbecher F, Na I (2023). “Single-cell clonal tracking of persistent T-cells in allogeneic hematopoietic stem cell
transplantation.” _Frontiers in Immunology_, 14. ISSN 1664-3224, doi:10.3389/fimmu.2023.1114368  [☞ Link](https://www.frontiersin.org/articles/10.3389/fimmu.2023.1114368)

Gressier E, Schulte-Schrepping J, Petrov L, Brumhard S, Stubbemann P, Hiller A, Obermayer B, Spitzer J, Kostevc T, Whitney PG, Bachem A, Odainic A,
van de Sandt C, Nguyen THO, Ashhurst T, Wilson K, Oates CVL, Gearing LJ, Meischel T, Hochheiser K, Greyer M, Clarke M, Kreutzenbeck M, Gabriel SS,
Kastenmüller W, Kurts C, Londrigan SL, Kallies A, Kedzierska K, Hertzog PJ, Latz E, Chen YE, Radford KJ, Chopin M, Schroeder J, Kurth F, Gebhardt T,
Sander LE, Sawitzki B, Schultze JL, Schmidt SV, Bedoui S (2023). “CD4+ T cell calibration of antigen-presenting cells optimizes antiviral CD8+ T cell
immunity.” _Nature Immunology_, 24(6), 979-990. ISSN 1529-2916, doi:10.1038/s41590-023-01517-x  [☞ Link](https://doi.org/10.1038/s41590-023-01517-x)

Loedige I, Baranovskii A, Mendonsa S, Dantsuji S, Popitsch N, Breimann L, Zerna N, Cherepanov V, Milek M, Ameres S, Chekulaeva M (2023). “mRNA stability
and m6A are major determinants of subcellular mRNA localization in neurons.” _Molecular Cell_, 83(15), 2709-2725.e10. ISSN 1097-2765,
doi:10.1016/j.molcel.2023.06.021  [☞ Link](https://www.sciencedirect.com/science/article/pii/S1097276523004689)

Boeddrich A, Haenig C, Neuendorf N, Blanc E, Ivanov A, Kirchner M, Schleumann P, Bayraktaroglu I, Richter M, Molenda CM, Sporbert A, Zenkner M, Schnoegl
S, Suenkel C, Schneider L, Rybak-Wolf A, Kochnowsky B, Byrne LM, Wild EJ, Nielsen JE, Dittmar G, Peters O, Beule D, Wanker EE (2023). “A proteomics
analysis of 5xFAD mouse brain regions reveals the lysosome-associated protein Arl8b as a candidate biomarker for Alzheimer's disease.” _Genome Med_,
15, 50. doi:10.1186/s13073-023-01206-2  [☞ Link](https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-023-01206-2)

de Bortoli T, Benary M, Horak P, Lamping M, Stintzing S, Tinhofer I, Leyvraz S, Schäfer R, Klauschen F, Keller U, Rieke DT, Jelas I (2023). “Tumour
mutational burden and survival with molecularly matched therapy.” _European Journal of Cancer_, 190, 112925. doi:10.1016/j.ejca.2023.05.013
 [☞ Link](https://www.sciencedirect.com/science/article/abs/pii/S0959804923002538)

Stellmach C, Sass J, Auber B, Boeker M, Wienker T, Heidel AJ, Benary M, Schumacher S, Ossowski S, Klauschen F, Möller Y, Schmutzler R, Ustjanzew A,
Werner P, Tomczak A, Hölter T, Thun S (2023). “Creation of a structured molecular genomics report for Germany as a local adaption of HL7’s Genomic
Reporting Implementation Guide.” _Journal of the American Medical Informatics Association_. ISSN 1527-974X, doi:10.1093/jamia/ocad061
 [☞ Link](https://doi.org/10.1093/jamia/ocad061)

Winkler W, Díaz CF, Blanc E, Napieczynska H, Langner P, Werner M, Walter B, Wollert-Wulf B, Yasuda T, Heuser A, Beule D, Mathas S, Anagnostopoulos I,
Rosenwald A, Rajewsky K, Janz M (2023). “Mouse models of human multiple myeloma subgroups.” _Proceedings of the National Academy of Sciences of the
United States of America_, 120, e2219439120. ISSN 1091-6490, doi:10.1073/pnas.2219439120  [☞ Link](https://doi.org/10.1073/pnas.2219439120)

Spranger L, Weiner J, Bredow J, Zeitz U, Grittner U, Boschmann M, Dickmann S, Stobaus N, Schwartzenberg JJ, Brachs M, Spranger J, Mai K (2023). “Thrifty
energy phenotype predicts weight regain in postmenopausal women with overweight or obesity and is related to FGFR1 signaling.” _Clinical Nutrition_,
42, 559-567. doi:10.1016/j.clnu.2023.02.020  [☞ Link](https://www.sciencedirect.com/science/article/abs/pii/S0261561423000560)

Freitag K, Eede P, Ivanov A, Sterczyk N, Schneeberger S, Borodina T, Sauer S, Beule D, Heppner F (2023). “Diverse but unique astrocytic phenotypes
during embryonic stem cell differentiation, culturing and development.” _Commun Biol_, 6, 40. doi:10.1038/s42003-023-04410-3
 [☞ Link](https://doi.org/10.1038/s42003-023-04410-3)

Nieminen M, Stolpe O, Kuhring M, Weiner J, Pett P, Beule D, Holtgrewe M (2023). “SODAR: managing multiomics study data and metadata.” _Gigascience_,
12, giad052. doi:10.1093/gigascience/giad052  [☞ Link](https://academic.oup.com/gigascience/article/doi/10.1093/gigascience/giad052/7232111)

Graff P, Wilzopolski J, Voss A, Blimkie TM, Weiner J3, Kershaw O, Panwar P, Hackett T, Brömme D, Loyal L, Thiel A, Beule D, Hancock RE, Gruber AD,
Bäumer W, Hedtrich S (2023). “Extracellular matrix remodeling in atopic dermatitis harnesses the onset of an asthmatic phenotype and is a potential
contributor to the atopic march.” _Journal of Investigative Dermatology_. doi:10.1016/j.jid.2023.09.278 

Müller R, Freitag-Wolf S, Weiner J3, Chopra A, Top T, Dommisch H, Schaefer A (2023). “Case-only design identifies interactions of genetic risk variants
at SIGLEC5 and PLG with the lncRNA CTD-2353F22.1 implying the importance of periodontal wound healing for disease aetiology.” _J Clin Periodontol_,
50, 90-101. doi:10.1111/jcpe.13712  [☞ Link](https://doi.org/10.1111/jcpe.13712)

Mothes R, Pascual-Reguant A, Koehler R, Liebeskind J, Liebheit A, Bauherr S, Philipsen L, Dittmayer C, Laue M, von Manitius R, Elezkurtaj S, Durek P,
Heinrich F, Heinz GA, Guerra GM, Obermayer B, Meinhardt J, Ihlow J, Radke J, Heppner FL, Enghard P, Stockmann H, Aschman T, Schneider J, Corman VM,
Sander LE, Mashreghi M, Conrad T, Hocke AC, Niesner RA, Radbruch H, Hauser AE (2023). “Distinct tissue niches direct lung immunopathology via CCL18 and
CCL21 in severe COVID-19.” _Nature Communications_, 14(1), 791. ISSN 2041-1723, doi:10.1038/s41467-023-36333-2
 [☞ Link](https://doi.org/10.1038/s41467-023-36333-2)

Rieke DT, Schröder S, Schafhausen P, Blanc E, Zuljan E, von der Emde B, Beule D, Keller U, Keilholz U, Klinghammer K (2023). “Targeted treatment in a
case series of AR+, HRAS/PIK3CA co-mutation salivary duct carcinoma.” _Frontiers in Oncology_, 13. ISSN 2234-943X, doi:10.3389/fonc.2023.1107134



### 2022


Holle J, Bartolomaeus H, Löber U, Behrens F, Bartolomaeus TUP, Anandakumar H, Wimmer MI, Vu DL, Kuhring M, Brüning U, Maifeld A, Geisberger S, Kempa S,
Schumacher F, Kleuser B, Bufler P, Querfeld U, Kitschke S, Engler D, Kuhrt LD, Drechsel O, Eckardt K, Forslund SK, Thürmer A, McParland V, Kirwan JA,
Wilck N, Müller D (2022). “Inflammation in Children with CKD Linked to Gut Dysbiosis and Metabolite Imbalance.” _Journal of the American Society of
Nephrology_, 33(12), 2259. ISSN 1046-6673, doi:10.1681/ASN.2022030378  [☞ Link](https://journals.lww.com/jasn/fulltext/2022/12000/Inflammation_in_Children_with_CKD_Linked_to_Gut.13.aspx)

Baumgardt M, Hülsemann M, Löwa A, Fatykhova D, Hoffmann K, Kessler M, Mieth M, Hellwig K, Frey D, Langenhagen A, Voss A, Obermayer B, Wyler E, Dökel S,
Gruber AD, Tölch U, Hippenstiel S, Hocke AC, Hönzke K (2022). “State-of-the-art analytical methods of viral infections in human lung organoids.” _PLOS
ONE_, 17(12), 1-8. doi:10.1371/journal.pone.0276115  [☞ Link](https://doi.org/10.1371/journal.pone.0276115)

Pennitz P, Kirsten H, Friedrich VD, Wyler E, Goekeri C, Obermayer B, Heinz GA, Mashreghi M, B"uttner M, Trimpert J, Landthaler M, Suttorp N, Hocke AC,
Hippenstiel S, T"onnies M, Scholz M, Kuebler WM, Witzenrath M, Hoenzke K, Nouailles G (2022). “A pulmonologist's guide to perform and analyse
cross-species single lung cell transcriptomics.” _European Respiratory Review_, 31(165). ISSN 0905-9180, doi:10.1183/16000617.0056-2022
 [☞ Link](https://err.ersjournals.com/content/31/165/220056)

Bertrams W, Hönzke K, Obermayer B, Tönnies M, Bauer TT, Schneider P, Neudecker J, Rückert JC, Stiewe T, Nist A, Eggeling S, Suttorp N, Wolff T,
Hippenstiel S, Schmeck B, Hocke AC (2022). “Transcriptomic comparison of primary human lung cells with lung tissue samples and the human A549 lung cell
line highlights cell type specific responses during infections with influenza A virus.” _Scientific Reports_, 12(1), 20608. ISSN 2045-2322,
doi:10.1038/s41598-022-24792-4  [☞ Link](https://doi.org/10.1038/s41598-022-24792-4)

Jarick K, Topczewska P, Jakob M, Yano H, Arifuzzaman M, Gao X, Boulekou S, Stokic-Trtica V, Leclère P, Preußer A, Rompe Z, Stamm A, Tsou A, Chu C,
Heinrich F, Guerra G, Durek P, Ivanov A, Beule D, Helfrich S, Duerr C, Kühl A, Stehle C, Romagnani C, Mashreghi M, Diefenbach A, Artis D, Klose C
(2022). “Non-redundant functions of group 2 innate lymphoid cells.” _Nature_, 611(7937), 794-800. doi:10.1038/s41586-022-05395-5
 [☞ Link](https://doi.org/10.1038/s41586-022-05395-5)

Ivanov A, Mattei D, Radscheit K, Compagnion A, Pett J, Herzel H, Paolicelli R, Piwecka M, Meyer U, Beule D (2022). “Analyses of circRNA Expression
throughout the Light-Dark Cycle Reveal a Strong Regulation of Cdr1as, Associated with Light Entrainment in the SCN.” _Int J Mol Sci_, 23.
doi:10.3390/ijms232012347  [☞ Link](https://doi.org/10.3390/ijms232012347)

Wirges A, Bunse M, Joedicke J, Blanc E, Gudipati V, Moles M, Shiku H, Beule D, Huppa J, Höpken U, Rehm A (2022). “EBAG9 silencing exerts an immune
checkpoint function without aggravating adverse effects.” _Mol Ther_, 30, 3358-3378. doi:10.1016/j.ymthe.2022.07.009
 [☞ Link](https://doi.org/10.1016/j.ymthe.2022.07.009)

Hönzke K, Obermayer B, Mache C, Fathykova D, Kessler M, Dökel S, Wyler E, Baumgardt M, Löwa A, Hoffmann K, Graff P, Schulze J, Mieth M, Hellwig K, Demir
Z, Biere B, Brunotte L, Mecate-Zambrano A, Bushe J, Dohmen M, Hinze C, Elezkurtaj S, Tönnies M, Bauer T, Eggeling S, Tran H, Schneider P, Neudecker J,
Rückert J, Schmidt-Ott K, Busch J, Klauschen F, Horst D, Radbruch H, Radke J, Heppner F, Corman V, Niemeyer D, Müller M, Goffinet C, Mothes R,
Pascual-Reguant A, Hauser A, Beule D, Landthaler M, Ludwig S, Suttorp N, Witzenrath M, Gruber A, Drosten C, Sander L, Wolff T, Hippenstiel S, Hocke A
(2022). “Human lungs show limited permissiveness for SARS-CoV-2 due to scarce ACE2 levels but virus-induced expansion of inflammatory macrophages.” _Eur
Respir J_, 60. doi:10.1183/13993003.02725-2021  [☞ Link](https://doi.org/10.1183/13993003.02725-2021)

Freitag K, Sterczyk N, Wendlinger S, Obermayer B, Schulz J, Farztdinov V, Mülleder M, Ralser M, Houtman J, Fleck L, Braeuning C, Sansevrino R, Hoffmann
C, Milovanovic D, Sigrist S, Conrad T, Beule D, Heppner F, Jendrach M (2022). “Spermidine reduces neuroinflammation and soluble amyloid beta in an
Alzheimer's disease mouse model.” _J Neuroinflammation_, 19, 172. doi:10.1186/s12974-022-02534-7  [☞ Link](https://doi.org/10.1186/s12974-022-02534-7)

Schalbetter S, von Arx A, Cruz-Ochoa N, Dawson K, Ivanov A, Mueller F, Lin H, Amport R, Mildenberger W, Mattei D, Beule D, Földy C, Greter M, Notter T,
Meyer U (2022). “Adolescence is a sensitive period for prefrontal microglia to act on cognitive development.” _Sci Adv_, 8, eabi6672.
doi:10.1126/sciadv.abi6672  [☞ Link](https://doi.org/10.1126/sciadv.abi6672)

Grunert C, Willimsky G, Peuker C, Rhein S, Hansmann L, Blankenstein T, Blanc E, Beule D, Keller U, Pezzutto A, Busse A (2022). “Isolation of
Neoantigen-Specific Human T Cell Receptors from Different Human and Murine Repertoires.” _Cancers (Basel)_, 14. doi:10.3390/cancers14071842
 [☞ Link](https://doi.org/10.3390/cancers14071842)

Dannebaum R, Suwalski P, Asgharian H, Du ZG, Lin H, Weiner J, Holtgrewe M, Thibeault C, Müller M, Wang X, Karadeniz Z, Saccomanno J, Doehn J, Hübner R,
Hinzmann B, Blüher A, Siemann S, Telman D, Suttorp N, Witzenrath M, Hippenstiel S, Skurk C, Poller W, Sander L, Beule D, Kurth F, Guettouche T,
Landmesser U, Berka J, Luong K, Pa-COVID SG, Rubelt F, Heidecker B (2022). “Highly multiplexed immune repertoire sequencing links multiple lymphocyte
classes with severity of response to COVID-19.” _EClinicalMedicine_, 48, 101438. doi:10.1016/j.eclinm.2022.101438
 [☞ Link](https://doi.org/10.1016/j.eclinm.2022.101438)

Horak P, Griffith M, Danos A, Pitel B, Madhavan S, Liu X, Chow C, Williams H, Carmody L, Barrow-Laing L, Rieke D, Kreutzfeldt S, Stenzinger A, Tamborero
D, Benary M, Rajagopal P, Ida C, Lesmana H, Satgunaseelan L, Merker J, Tolstorukov M, Campregher P, Warner J, Rao S, Natesan M, Shen H, Venstrom J, Roy
S, Tao K, Kanagal-Shamanna R, Xu X, Ritter D, Pagel K, Krysiak K, Dubuc A, Akkari Y, Li X, Lee J, King I, Raca G, Wagner A, Li M, Plon S, Kulkarni S,
Griffith O, Chakravarty D, Sonkin D (2022). “Standards for the classification of pathogenicity of somatic variants in cancer (oncogenicity): Joint
recommendations of Clinical Genome Resource (ClinGen), Cancer Genomics Consortium (CGC), and Variant Interpretation for Cancer Consortium (VICC).”
_Genet Med_, 24, 986-998. doi:10.1016/j.gim.2022.01.001  [☞ Link](https://doi.org/10.1016/j.gim.2022.01.001)

Weiner J, Obermayer B, Beule D (2022). “Venn Diagrams May Indicate Erroneous Statistical Reasoning in Transcriptomics.” _Front Genet_, 13, 818683.
doi:10.3389/fgene.2022.818683  [☞ Link](https://doi.org/10.3389/fgene.2022.818683)

Gloaguen Y, Kirwan J, Beule D (2022). “Deep Learning-Assisted Peak Curation for Large-Scale LC-MS Metabolomics.” _Anal Chem_, 94, 4930-4937.
doi:10.1021/acs.analchem.1c02220  [☞ Link](https://doi.org/10.1021/acs.analchem.1c02220)

Chopra A, Mueller R, Weiner J3, Rosowski J, Dommisch H, Grohmann E, Schaefer A (2022). “BACH1 Binding Links the Genetic Risk for Severe Periodontitis
with ST8SIA1.” _J Dent Res_, 101, 93-101. doi:10.1177/00220345211017510  [☞ Link](https://doi.org/10.1177/00220345211017510)

Bischoff P, Trinks A, Wiederspahn J, Obermayer B, Pett J, Jurmeister P, Elsner A, Dziodzio T, Rückert J, Neudecker J, Falk C, Beule D, Sers C, Morkel M,
Horst D, Klauschen F, Blüthgen N (2022). “The single-cell transcriptional landscape of lung carcinoid tumors.” _Int J Cancer_, 150, 2058-2071.
doi:10.1002/ijc.33995  [☞ Link](https://doi.org/10.1002/ijc.33995)

Georg P, Astaburuaga-García R, Bonaguro L, Brumhard S, Michalick L, Lippert L, Kostevc T, Gäbel C, Schneider M, Streitz M, Demichev V, Gemünd I, Barone
M, Tober-Lau P, Helbig E, Hillus D, Petrov L, Stein J, Dey H, Paclik D, Iwert C, Mülleder M, Aulakh S, Djudjaj S, Bülow R, Mei H, Schulz A, Thiel A,
Hippenstiel S, Saliba A, Eils R, Lehmann I, Mall M, Stricker S, Röhmel J, Corman V, Beule D, Wyler E, Landthaler M, Obermayer B, von Stillfried S, Boor
P, Demir M, Wesselmann H, Suttorp N, Uhrig A, Müller-Redetzky H, Nattermann J, Kuebler W, Meisel C, Ralser M, Schultze J, Aschenbrenner A, Thibeault C,
Kurth F, Sander L, Blüthgen N, Sawitzki B, PA-COVID-19 SG (2022). “Complement activation induces excessive T cell cytotoxicity in severe COVID-19.”
_Cell_, 185(3), 493-512.e25. ISSN 0092-8674, doi:10.1016/j.cell.2021.12.040  [☞ Link](https://doi.org/10.1016/j.cell.2021.12.040)

Zinnall U, Milek M, Minia I, Vieira-Vieira C, Müller S, Mastrobuoni G, Hazapis O, Del GS, Schwefel D, Bley N, Voigt F, Chao J, Kempa S, Hüttelmaier S,
Selbach M, Landthaler M (2022). “HDLBP binds ER-targeted mRNAs by multivalent interactions to promote protein synthesis of transmembrane and secreted
proteins.” _Nat Commun_, 13(1), 2727. doi:10.1038/s41467-022-30322-7  [☞ Link](https://doi.org/10.1038/s41467-022-30322-7)

Dörr D, Obermayer B, Weiner J, Zimmermann K, Anania C, Wagner L, Lyras E, Sapozhnikova V, Lara-Astiaso D, Prósper F, Lang R, Lupiáñez D, Beule D, Höpken
U, Leutz A, Mildner A (2022). “C/EBPβ regulates lipid metabolism and Pparg isoform 2 expression in alveolar macrophages.” _Sci Immunol_, 7(75),
eabj0140. doi:10.1126/sciimmunol.abj0140  [☞ Link](https://doi.org/10.1126/sciimmunol.abj0140)

Kedziora S, Obermayer B, Sugulle M, Herse F, Kräker K, Haase N, Langmia I, Müller D, Staff A, Beule D, Dechend R (2022). “Placental Transcriptome
Profiling in Subtypes of Diabetic Pregnancies Is Strongly Confounded by Fetal Sex.” _Int J Mol Sci_, 23(23). ISSN 1422-0067, doi:10.3390/ijms232315388
 [☞ Link](https://doi.org/10.3390/ijms232315388)

Pennitz P, Kirsten H, Wyler E, Goekeri C, Obermayer B, Heinz GA, Mashreghi MF, Trimpert J, Landthaler M, Suttorp N, Hocke AC, Hippenstiel S, Tönnies M,
Scholz M, Kuebler WM, Witzenrath M, Hoenzke K, Nouailles G (2022). “Integrational approaches for cross-species analysis of lung pathologies at
single-cell resolution.” _Pneumologie_, 76(S 01), FV 285. FV 285.

Hoffmann K, Obermayer B, Hönzke K, Fatykhova D, Demir Z, Löwa A, Alves L, Wyler E, Lopez-Rodriguez E, Mieth M, Baumgardt M, Hoppe J, Firsching T,
Tönnies M, Bauer T, Eggeling S, Tran H, Schneider P, Neudecker J, Rückert J, Gruber A, Ochs M, Landthaler M, Beule D, Suttorp N, Hippenstiel S, Hocke A,
Kessler M (2022). “Human alveolar progenitors generate dual lineage bronchioalveolar organoids.” _Commun Biol_, 5(1), 875. ISSN 2399-3642,
doi:10.1038/s42003-022-03828-5  [☞ Link](https://doi.org/10.1038/s42003-022-03828-5)

Neitzel H, Varon R, Chughtai S, Dartsch J, Dutrannoy-Tönsing V, Nürnberg P, Nürnberg G, Schweiger M, Digweed M, Hildebrand G, Hackmann K, Holtgrewe M,
Sarioglu N, Schulze B, Horn D, Sperling K (2022). “Transmission ratio distortion of mutations in the master regulator of centriole biogenesis PLK4.”
_Hum Genet_, 141(11), 1785-1794. doi:10.1007/s00439-022-02461-w  [☞ Link](https://doi.org/10.1007/s00439-022-02461-w)

Foddis M, Blumenau S, Holtgrewe M, Paquette K, Westra K, Alonso I, Macario M, Morgadinho A, Velon A, Santo G, Santana I, Mönkäre S, Kuuluvainen L,
Schleutker J, Pöyhönen M, Myllykangas L, Pavlovic A, Kostic V, Dobricic V, Lohmann E, Hanagasi H, Santos M, Guven G, Bilgic B, Bras J, Beule D, Dirnagl
U, Guerreiro R, Sassi C (2022). “TREX1 p.A129fs and p.Y305C variants in a large multi-ethnic cohort of CADASIL-like unrelated patients.”
doi:10.1016/j.neurobiolaging.2022.11.013  [☞ Link](https://doi.org/10.1016/j.neurobiolaging.2022.11.013)

Boschann F, Moreno D, Mensah M, Sczakiel H, Skipalova K, Holtgrewe M, Mundlos S, Fischer-Zirnsak B (2022). “Xq27.1 palindrome mediated interchromosomal
insertion likely causes familial congenital bilateral laryngeal abductor paralysis (Plott syndrome).” _J Hum Genet_, 67(7), 405-410.
doi:10.1038/s10038-022-01018-z  [☞ Link](https://doi.org/10.1038/s10038-022-01018-z)

Seidel F, Laser K, Klingel K, Dartsch J, Theisen S, Pickardt T, Holtgrewe M, Gärtner A, Berger F, Beule D, Milting H, Schubert S, Klaassen S, Kühnisch J
(2022). “Pathogenic Variants in Cardiomyopathy Disorder Genes Underlie Pediatric Myocarditis-Further Impact of Heterozygous Immune Disorder Gene
Variants?” _J Cardiovasc Dev Dis_, 9(7), 216. doi:10.3390/jcdd9070216  [☞ Link](https://doi.org/10.3390/jcdd9070216)

May V, Koch L, Fischer-Zirnsak B, Horn D, Gehle P, Kornak U, Beule D, Holtgrewe M (2022). “ClearCNV: CNV calling from NGS panel data in the presence of
ambiguity and noise.” _Bioinformatics_, 38(16), 3871-3876. doi:10.1093/bioinformatics/btac418  [☞ Link](https://doi.org/10.1093/bioinformatics/btac418)

Schöpflin R, Melo U, Moeinzadeh H, Heller D, Laupert V, Hertzberg J, Holtgrewe M, Alavi N, Klever M, Jungnitsch J, Comak E, Türkmen S, Horn D, Duffourd
Y, Faivre L, Callier P, Sanlaville D, Zuffardi O, Tenconi R, Kurtas N, Giglio S, Prager B, Latos-Bielenska A, Vogel I, Bugge M, Tommerup N, Spielmann M,
Vitobello A, Kalscheuer V, Vingron M, Mundlos S (2022). “Integration of Hi-C with short and long-read genome sequencing reveals the structure of
germline rearranged genomes.” _Nat Commun_, 13(1), 6470. doi:10.1038/s41467-022-34053-7  [☞ Link](https://doi.org/10.1038/s41467-022-34053-7)

Rieke D, de BT, Horak P, Lamping M, Benary M, Jelas I, Rüter G, Berger J, Zettwitz M, Kagelmann N, Kind A, Fabian F, Beule D, Glimm H, Brors B,
Stenzinger A, Fröhling S, Keilholz U (2022). “Feasibility and outcome of reproducible clinical interpretation of high-dimensional molecular data: a
comparison of two molecular tumor boards.” _BMC Med_, 20, 367. ISSN 17417015, doi:10.1186/s12916-022-02560-5
 [☞ Link](https://doi.org/10.1186/s12916-022-02560-5)

Hinze C, Kocks C, Leiz J, Karaiskos N, Boltengagen A, Cao S, Skopnik C, Klocke J, Hardenberg J, Stockmann H, Gotthardt I, Obermayer B, Haghverdi L,
Wyler E, Landthaler M, Bachmann S, Hocke A, Corman V, Busch J, Schneider W, Himmerkus N, Bleich M, Eckardt K, Enghard P, Rajewsky N, Schmidt-Ott K
(2022). “Single-cell transcriptomics reveals common epithelial response patterns in human acute kidney injury.” _Genome Med_, 14, 103.
doi:10.1186/s13073-022-01108-9  [☞ Link](https://doi.org/10.1186/s13073-022-01108-9)

Richter G, Wagner G, Reichenmiller K, Staufenbiel I, Martins O, Löscher B, Holtgrewe M, Jepsen S, Dommisch H, Schaefer A (2022). “Exome Sequencing of 5
Families with Severe Early-Onset Periodontitis.” _J Dent Res_, 101(2), 151-157. doi:10.1177/00220345211029266
 [☞ Link](https://doi.org/10.1177/00220345211029266)

Coutelier M, Holtgrewe M, Jäger M, Flöttman R, Mensah M, Spielmann M, Krawitz P, Horn D, Beule D, Mundlos S (2022). “Combining callers improves the
detection of copy number variants from whole-genome sequencing.” _Eur J Hum Genet_, 30(2), 178-186. doi:10.1038/s41431-021-00983-x
 [☞ Link](https://doi.org/10.1038/s41431-021-00983-x)

Vogt G, Verheyen S, Schwartzmann S, Ehmke N, Potratz C, Schwerin-Nagel A, Plecko B, Holtgrewe M, Seelow D, Blatterer J, Speicher M, Kornak U, Horn D,
Mundlos S, Fischer-Zirnsak B, Boschann F (2022). “Biallelic truncating variants in ATP9A cause a novel neurodevelopmental disorder involving postnatal
microcephaly and failure to thrive.” _J Med Genet_, 59(7), 662-668. doi:10.1136/jmedgenet-2021-107843  [☞ Link](https://doi.org/10.1136/jmedgenet-2021-107843)

Kossen T, Hirzel MA, Madai VI, Boenisch F, Hennemuth A, Hildebrand K, Pokutta S, Sharma K, Hilbert A, Sobesky J, Galinovic I, Khalil AA, Fiebach JB and Frey D. Toward Sharing Brain Images: Differentially Private TOF-MRA Images With Segmentation Labels Using Generative Adversarial Networks. *Frontiers in
Artificial Intelligence*. 5 (2022). issn: 2624-8212. [doi: 10.3389/frai.2022.813842](https://doi.org/10.3389/frai.2022.813842)

### 2021


Schinke C, Fernandez VV, Ivanov A, Peng Y, Körtvelyessy P, Nolte L, Huehnchen P, Beule D, Stachelscheid H, Boehmerle W, Endres M (2021). “Dataset for:
Modeling chemotherapy induced neurotoxicity with human induced pluripotent stem cell (iPSC)-derived sensory neurons.” _Data Brief_, 38, 107320.
doi:10.1016/j.dib.2021.107320  [☞ Link](https://doi.org/10.1016/j.dib.2021.107320)

Weiner J, Suwalski P, Holtgrewe M, Rakitko A, Thibeault C, Müller M, Patriki D, Quedenau C, Krüger U, Ilinsky V, Popov I, Balnis J, Jaitovich A, Helbig
E, Lippert L, Stubbemann P, Real L, Macías J, Pineda J, Fernandez-Fuertes M, Wang X, Karadeniz Z, Saccomanno J, Doehn J, Hübner R, Hinzmann B, Salvo M,
Blueher A, Siemann S, Jurisic S, Beer J, Rutishauser J, Wiggli B, Schmid H, Danninger K, Binder R, Corman V, Mühlemann B, Arjun AR, Fragiadakis G, Mick
E, Comet C, Calfee C, Erle D, Hendrickson C, Kangelaris K, Krummel M, Woodruff P, Langelier C, Venkataramani U, García F, Zyla J, Drosten C, Alice B,
Jones T, Suttorp N, Witzenrath M, Hippenstiel S, Zemojtel T, Skurk C, Poller W, Borodina T, Pa-Covid S, Ripke S, Sander L, Beule D, Landmesser U,
Guettouche T, Kurth F, Heidecker B (2021). “Increased risk of severe clinical course of COVID-19 in carriers of HLA-C\04:01.” _EClinicalMedicine_,
40, 101099. doi:10.1016/j.eclinm.2021.101099  [☞ Link](https://doi.org/10.1016/j.eclinm.2021.101099)

Domaszewska T, Zyla J, Otto R, Kaufmann S, Weiner J (2021). “Gene Set Enrichment Analysis Reveals Individual Variability in Host Responses in
Tuberculosis Patients.” _Front Immunol_, 12, 694680. doi:10.3389/fimmu.2021.694680  [☞ Link](https://doi.org/10.3389/fimmu.2021.694680)

Nouailles G, Wyler E, Pennitz P, Postmus D, Vladimirova D, Kazmierski J, Pott F, Dietert K, Muelleder M, Farztdinov V, Obermayer B, Wienhold S,
Andreotti S, Hoefler T, Sawitzki B, Drosten C, Sander LE, Suttorp N, Ralser M, Beule D, Gruber AD, Goffinet C, Landthaler M, Trimpert J, Witzenrath M
(2021). “Single-cell-sequencing in SARS-COV-2-infected hamsters sheds light on endothelial cell involvement in COVID-19.” _European Respiratory
Journal_. doi:10.1183/13993003.congress-2021.PA2355  [☞ Link](https://erj.ersjournals.com/content/58/suppl_65/PA2355.abstract)

Bischoff P, Trinks A, Obermayer B, Pett J, Wiederspahn J, Uhlitz F, Liang X, Lehmann A, Jurmeister P, Elsner A, Dziodzio T, Rückert J, Neudecker J, Falk
C, Beule D, Sers C, Morkel M, Horst D, Blüthgen N, Klauschen F (2021). “Single-cell RNA sequencing reveals distinct tumor microenvironmental patterns in
lung adenocarcinoma.” _Oncogene_, 40, 6748-6758. doi:10.1038/s41388-021-02054-3  [☞ Link](https://doi.org/10.1038/s41388-021-02054-3)

Uhlitz F, Bischoff P, Peidli S, Sieber A, Trinks A, Lüthen M, Obermayer B, Blanc E, Ruchiy Y, Sell T, Mamlouk S, Arsie R, Wei T, Klotz-Noack K, Schwarz
R, Sawitzki B, Kamphues C, Beule D, Landthaler M, Sers C, Horst D, Blüthgen N, Morkel M (2021). “Mitogen-activated protein kinase activity drives cell
trajectories in colorectal cancer.” _EMBO Mol Med_, 13, e14123. doi:10.15252/emmm.202114123  [☞ Link](https://doi.org/10.15252/emmm.202114123)

Nouailles G, Wyler E, Pennitz P, Postmus D, Vladimirova D, Kazmierski J, Pott F, Dietert K, Muelleder M, Farztdinov V, Obermayer B, Wienhold S,
Andreotti S, Hoefler T, Sawitzki B, Drosten C, Sander L, Suttorp N, Ralser M, Beule D, Gruber A, Goffinet C, Landthaler M, Trimpert J, Witzenrath M
(2021). “Temporal omics analysis in Syrian hamsters unravel cellular effector responses to moderate COVID-19.” _Nat Commun_, 12, 4869.
doi:10.1038/s41467-021-25030-7  [☞ Link](https://doi.org/10.1038/s41467-021-25030-7)

Paterson S, Kar S, Ung S, Gardener Z, Bergstrom E, Ascough S, Kalyan M, Zyla J, Maertzdorf J, Mollenkopf H, Weiner J, Jozwik A, Jarvis H, Jha A,
Nicholson B, Veldman T, Woods C, Mallia P, Kon O, Kaufmann S, Openshaw P, Chiu C (2021). “Innate-like Gene Expression of Lung-Resident Memory CD8(+) T
Cells during Experimental Human Influenza: A Clinical Study.” _Am J Respir Crit Care Med_, 204, 826-841. doi:10.1164/rccm.202103-0620OC
 [☞ Link](https://doi.org/10.1164/rccm.202103-0620OC)

Rajasekaran S, Siddiqui J, Rakijas J, Nicolay B, Lin C, Khan E, Patel R, Morris R, Wyler E, Boukhali M, Balasubramanyam J, Ranjith KR, van Rechem C,
Vogel C, Elchuri S, Landthaler M, Obermayer B, Haas W, Dyson N, Miles W (2021). “Integrated multi-omics analysis of RB-loss identifies widespread
cellular programming and synthetic weaknesses.” _Commun Biol_, 4, 977. doi:10.1038/s42003-021-02495-2  [☞ Link](https://doi.org/10.1038/s42003-021-02495-2)

Schinke C, Fernandez VV, Ivanov A, Peng Y, Körtvelyessy P, Nolte L, Huehnchen P, Beule D, Stachelscheid H, Boehmerle W, Endres M (2021). “Modeling
chemotherapy induced neurotoxicity with human induced pluripotent stem cell (iPSC) -derived sensory neurons.” _Neurobiol Dis_, 155, 105391.
doi:10.1016/j.nbd.2021.105391  [☞ Link](https://doi.org/10.1016/j.nbd.2021.105391)

Pei G, Zyla J, He L, Moura-Alves P, Steinle H, Saikali P, Lozza L, Nieuwenhuizen N, Weiner J, Mollenkopf H, Ellwanger K, Arnold C, Duan M, Dagil Y,
Pashenkov M, Boneca I, Kufer T, Dorhoi A, Kaufmann S (2021). “Cellular stress promotes NOD1/2-dependent inflammation via the endogenous metabolite
sphingosine-1-phosphate.” _EMBO J_, 40, e106272. doi:10.15252/embj.2020106272  [☞ Link](https://doi.org/10.15252/embj.2020106272)

Herda S, Heimann A, Obermayer B, Ciraolo E, Althoff S, Ruß J, Grunert C, Busse A, Bullinger L, Pezzutto A, Blankenstein T, Beule D, Na I (2021).
“Long-term in vitro expansion ensures increased yield of central memory T cells as perspective for manufacturing challenges.” _Int J Cancer_, 148,
3097-3110. doi:10.1002/ijc.33523  [☞ Link](https://doi.org/10.1002/ijc.33523)

Kraushar M, Krupp F, Harnett D, Turko P, Ambrozkiewicz M, Sprink T, Imami K, Günnigmann M, Zinnall U, Vieira-Vieira C, Schaub T, Münster-Wandowski A,
Bürger J, Borisova E, Yamamoto H, Rasin M, Ohler U, Beule D, Mielke T, Tarabykin V, Landthaler M, Kramer G, Vida I, Selbach M, Spahn C (2021). “Protein
Synthesis in the Developing Neocortex at Near-Atomic Resolution Reveals Ebp1-Mediated Neuronal Proteostasis at the 60S Tunnel Exit.” _Mol Cell_, 81,
304-322.e16. doi:10.1016/j.molcel.2020.11.037  [☞ Link](https://doi.org/10.1016/j.molcel.2020.11.037)

Heyckendorf J, Marwitz S, Reimann M, Avsar K, DiNardo A, Günther G, Hoelscher M, Ibraim E, Kalsdorf B, Kaufmann S, Kontsevaya I, van Leth F, Mandalakas
A, Maurer F, Müller M, Nitschkowski D, Olaru I, Popa C, Rachow A, Rolling T, Rybniker J, Salzer H, Sanchez-Carballo P, Schuhmann M, Schaub D, Spinu V,
Suárez I, Terhalle E, Unnewehr M, Weiner J3, Goldmann T, Lange C (2021). “Prediction of anti-tuberculosis treatment duration based on a 22-gene
transcriptomic model.” _Eur Respir J_, 58. doi:10.1183/13993003.03492-2020  [☞ Link](https://doi.org/10.1183/13993003.03492-2020)

Bao X, Weiner J3, Meckes O, Dommisch H, Schaefer A (2021). “Entamoeba gingivalis Exerts Severe Pathogenic Effects on the Oral Mucosa.” _J Dent Res_,
100, 771-776. doi:10.1177/00220345211004498  [☞ Link](https://doi.org/10.1177/00220345211004498)

Eckold C, Kumar V, Weiner J, Alisjahbana B, Riza A, Ronacher K, Coronel J, Kerry-Barnard S, Malherbe S, Kleynhans L, Stanley K, Ruslami R, Ioana M,
Ugarte-Gil C, Walzl G, van Crevel R, Wijmenga C, Critchley J, Dockrell H, Cliff J, TANDEM c (2021). “Impact of Intermediate Hyperglycemia and Diabetes
on Immune Dysfunction in Tuberculosis.” _Clin Infect Dis_, 72, 69-78. doi:10.1093/cid/ciaa751  [☞ Link](https://doi.org/10.1093/cid/ciaa751)

Wyler E, Mösbauer K, Franke V, Diag A, Gottula L, Arsiè R, Klironomos F, Koppstein D, Hönzke K, Ayoub S, Buccitelli C, Hoffmann K, Richter A, Legnini I,
Ivanov A, Mari T, Del GS, Papies J, Praktiknjo S, Meyer T, Müller M, Niemeyer D, Hocke A, Selbach M, Akalin A, Rajewsky N, Drosten C, Landthaler M
(2021). “Transcriptomic profiling of SARS-CoV-2 infected human cell lines identifies HSP90 as target for COVID-19 therapy.” _iScience_, 24, 102151.
doi:10.1016/j.isci.2021.102151  [☞ Link](https://doi.org/10.1016/j.isci.2021.102151)

Mölder F, Jablonski K, Letcher B, Hall M, Tomkins-Tinch C, Sochat V, Forster J, Lee S, Twardziok S, Kanitz A, Wilm A, Holtgrewe M, Rahmann S, Nahnsen S,
Köster J (2021). “Sustainable data analysis with Snakemake.” _F1000Res_, 10, 33. doi:10.12688/f1000research.29032.2
 [☞ Link](https://doi.org/10.12688/f1000research.29032.2)

Messerschmidt C, Foddis M, Blumenau S, Müller S, Bentele K, Holtgrewe M, Kun-Rodrigues C, Alonso I, do CMM, Morgadinho A, Velon A, Santo G, Santana I,
Mönkäre S, Kuuluvainen L, Schleutker J, Pöyhönen M, Myllykangas L, Senatore A, Berchtold D, Winek K, Meisel A, Pavlovic A, Kostic V, Dobricic V, Lohmann
E, Hanagasi H, Guven G, Bilgic B, Bras J, Guerreiro R, Beule D, Dirnagl U, Sassi C (2021). “PHACTR1 genetic variability is not critical in small vessel
ischemic disease patients and PcomA recruitment in C57BL/6J mice.” _Sci Rep_, 11(1), 6072. doi:10.1038/s41598-021-84919-x
 [☞ Link](https://doi.org/10.1038/s41598-021-84919-x)

Zarate Y, Uehara T, Abe K, Oginuma M, Harako S, Ishitani S, Lehesjoki A, Bierhals T, Kloth K, Ehmke N, Horn D, Holtgrewe M, Anderson K, Viskochil D,
Edgar-Zarate C, Sacoto M, Schnur R, Morrow M, Sanchez-Valle A, Pappas J, Rabin R, Muona M, Anttonen A, Platzer K, Luppe J, Gburek-Augustat J, Kaname T,
Okamoto N, Mizuno S, Kaido Y, Ohkuma Y, Hirose Y, Ishitani T, Kosaki K (2021). “CDK19-related disorder results from both loss-of-function and
gain-of-function de novo missense variants.” _Genet Med_, 23(6), 1050-1057. doi:10.1038/s41436-020-01091-9
 [☞ Link](https://doi.org/10.1038/s41436-020-01091-9)

Vogt G, El CN, Herczegfalvi Á, Kölbel H, Lekaj A, Schara U, Holtgrewe M, Krause S, Horvath R, Schuelke M, Hübner C, Mundlos S, Roos A, Lochmüller H,
Karcagi V, Kornak U, Fischer-Zirnsak B (2021). “Expanding the clinical and molecular spectrum of ATP6V1A related metabolic cutis laxa.” _J Inherit Metab
Dis_, 44(4), 972-986. doi:10.1002/jimd.12341  [☞ Link](https://doi.org/10.1002/jimd.12341)

Yildirim F, Foddis M, Blumenau S, Müller S, Kajetan B, Holtgrewe M, Kola V, Beule D, Sassi C (2021). “Shared and oppositely regulated transcriptomic
signatures in Huntington's disease and brain ischemia confirm known and unveil novel potential neuroprotective genes.” _Neurobiol Aging_, 104,
122.e1-122.e17. doi:10.1016/j.neurobiolaging.2021.03.001  [☞ Link](https://doi.org/10.1016/j.neurobiolaging.2021.03.001)

Elsner J, Mensah M, Holtgrewe M, Hertzberg J, Bigoni S, Busche A, Coutelier M, de SD, Elçioglu N, Filges I, Gerkes E, Girisha K, Graul-Neumann L,
Jamsheer A, Krawitz P, Kurth I, Markus S, Megarbane A, Reis A, Reuter M, Svoboda D, Teller C, Tuysuz B, Türkmen S, Wilson M, Woitschach R, Vater I,
Caliebe A, Hülsemann W, Horn D, Mundlos S, Spielmann M (2021). “Genome sequencing in families with congenital limb malformations.” _Hum Genet_,
140(8), 1229-1239. doi:10.1007/s00439-021-02295-y  [☞ Link](https://doi.org/10.1007/s00439-021-02295-y)

Seidel F, Holtgrewe M, Al-Wakeel-Marquard N, Opgen-Rhein B, Dartsch J, Herbst C, Beule D, Pickardt T, Klingel K, Messroghli D, Berger F, Schubert S,
Kühnisch J, Klaassen S (2021). “Pathogenic Variants Associated With Dilated Cardiomyopathy Predict Outcome in Pediatric Myocarditis.” _Circ Genom Precis
Med_, 14(4), e003250. doi:10.1161/CIRCGEN.120.003250  [☞ Link](https://doi.org/10.1161/CIRCGEN.120.003250)

Melo U, Piard J, Fischer-Zirnsak B, Klever M, Schöpflin R, Mensah M, Holtgrewe M, Arbez-Gindre F, Martin A, Guigue V, Gaillard D, Landais E, Roze V,
Kremer V, Ramanah R, Cabrol C, Harms F, Kornak U, Spielmann M, Mundlos S, van Maldergem L (2021). “Complete lung agenesis caused by complex genomic
rearrangements with neo-TAD formation at the SHH locus.” _Hum Genet_, 140(10), 1459-1469. doi:10.1007/s00439-021-02344-6
 [☞ Link](https://doi.org/10.1007/s00439-021-02344-6)

Sczakiel H, Hülsemann W, Holtgrewe M, Abad-Perez A, Elsner J, Schwartzmann S, Horn D, Spielmann M, Mundlos S, Mensah M (2021). “GLI3 variants causing
isolated polysyndactyly are not restricted to the protein's C-terminal third.” _Clin Genet_, 100(6), 758-765. doi:10.1111/cge.14059
 [☞ Link](https://doi.org/10.1111/cge.14059)

Li, N., Hollunder, B., Baldermann, J. C., Kibleur, A., Treu, S., Akram, H., Al-Fatly, B., Strange, B. A., Barcia, J. A., Zrinzo, L., Joyce, E. M., Chabardes, S., Visser-Vandewalle, V., Polosan, M., Kuhn, J., Kühn, A. A., & Horn, A. (2021). A Unified Functional Network Target for Deep Brain Stimulation in Obsessive-Compulsive Disorder. Biological Psychiatry. [doi: 10.1016/j.biopsych.2021.04.006](https://doi.org/10.1016/j.biopsych.2021.04.006)

Bressem KK, Vahldiek JL, Adams L, Niehues SM, Haibel H, Rodriguez VR, Torgutalp M, Protopopov M, Proft F, Rademacher J, Sieper J, Rudwaleit M, Hamm B, Makowski MR, Hermann KG, Poddubnyy D. 
Deep learning for detection of radiographic sacroiliitis: achieving expert-level performance. 
Arthritis Res Ther. 2021 Apr 8;23(1):106. [doi: 10.1186/s13075-021-02484-0](https://doi.org/10.1186/s13075-021-02484-0)

Kossen T, Subramaniam P, Madai VI, Hennemuth A, Hildebrand K, Hilbert A, Sobesky J, Livne M, Galinovic I, Khalil AA, Fiebach JB, Frey D. 
Synthesizing anonymized and labeled TOF-MRA patches for brain vessel segmentation using generative adversarial networks. 
*Computers in Biology and Medicine*. 2021 Apr 131,104254. [doi: 10.1016/j.compbiomed.2021.104254](https://doi.org/10.1016/j.compbiomed.2021.104254)

Paraskevopoulou S., Käfer S., Zirkel F., Donath A., Petersen M., Liu S., Zhou X., Drosten C., Misof B., Junglen S. (2021). 
"Viromics of extant insect orders unveil the evolution of the flavi-like superfamily."
*Virus Evolution*
2021 Mar 30. 
[doi: 10.1093/ve/veab030](https://doi.org/10.1093/ve/veab030)

Thomas Krannich, W Timothy J White, Sebastian Niehus, Guillaume Holley, Bjarni V Halldórsson, Birte Kehr, Population-scale detection of non-reference sequence variants using colored de Bruijn graphs, Bioinformatics, 2021, btab749, [doi: 10.1093/bioinformatics/btab749](https://doi.org/10.1093/bioinformatics/btab749)

Julia Markowski, Rieke Kempfer, Alexander Kukalev, Ibai Irastorza-Azcarate, Gesa Loof, Birte Kehr, Ana Pombo, Sven Rahmann, Roland F Schwarz, GAMIBHEAR: whole-genome haplotype reconstruction from Genome Architecture Mapping data, Bioinformatics, Volume 37, Issue 19, 1 October 2021, Pages 3128–3135. [doi: 10.1093/bioinformatics/btab238](https://doi.org/10.1093/bioinformatics/btab238)

### 2020

Krützfeldt LM, Schubach M, Kircher M. The impact of different negative training data on regulatory sequence predictions. PLoS One. 2020 Dec 1;15(12):e0237412. [doi: 10.1371/journal.pone.0237412](https://doi.org/10.1371/journal.pone.0237412).

Klotz-Noack K, Klinger B, Rivera M, Bublitz N, Uhlitz F, Riemer P, Lüthen M, Sell T, Kasack K, Gastl B, Ispasanie SSS, Simon T, Janssen N, Schwab M, Zuber J, Horst D, Blüthgen N, Schäfer R, Morkel M, Sers C. SFPQ Depletion Is Synthetically Lethal with BRAFV600E in Colorectal Cancer Cells. Cell Rep. 2020 Sep 22;32(12):108184. [doi: 10.1016/j.celrep.2020.108184](https://doi.org/10.1016/j.celrep.2020.108184).

Kleinert, P., Martin, B., & Kircher, M. (2020).
"HemoMIPs—Automated analysis and result reporting pipeline for targeted sequencing data."
*PLOS Computational Biology*,
16(6), e1007956.
[doi:  10.1371/journal.pcbi.1007956](https://doi.org/10.1371/journal.pcbi.1007956)

Ehmke, N.; Cusmano-Ozog, K.; Koenig, R.; Holtgrewe, M.; Nur, B.; Mihci, E.; Babcock, H.; Gonzaga-Jauregui, C.; Overton, J. D.; Xiao, J.; et al. Biallelic Variants in KYNU Cause a Multisystemic Syndrome with Hand Hyperphalangism. *Bone* 2020, 115219.
[doi: 10.1016/j.bone.2019.115219](https://doi.org/10.1016/j.bone.2019.115219).

Niehus, S.; Jónsson, H.; Schönberger, J.; Björnsson, E.; Beyter, D.; Eggertsson, H.P.; Sulem, P.; Stefánsson, K.; Halldórsson, B.V.; Kehr, B. PopDel identifies medium-size deletions jointly in tens of thousands of genomes.
*bioRxiv* 2020, 10.1101/740225
[doi: 10.1101/740225](https://doi.org/10.1101/740225)

Gordon, M. G., Inoue, F., Martin, B., Schubach, M., Agarwal, V., Whalen, S., ... & Kreimer, A. (2020).
"lentiMPRA and MPRAflow for high-throughput functional characterization of gene regulatory elements."
*Nature Protocols*, 15(8), 2387-2412.
[doi: 10.1038/s41596-020-0333-5](https://doi.org/10.1038/s41596-020-0333-5)

Paraskevopoulou S., Pirzer F., Goldmann N., Schmid J., Corman V.M., Gottula L.T.,Schroeder S., Rasche A., Muth D., Drexler J.F., Heni A.C., Eibner G.J., Page R.A., Jones T.C., MüllerM.A., Sommer S., Glebe D., and Drosten C. (2020).
"Mammalian deltavirus without hepadnavirus coinfection in the neotropical rodent *Proechimys semispinosus*."
*Proceedings of the National Academy of Sciences*
2020 Jul 28;117(30):17977-17983.
[doi: 10.1073/pnas.2006750117](https://doi.org/10.1073/pnas.2006750117).


Mattei D, Ivanov A, van Oostrum M, Pantelyushin S, Richetto J, Mueller F, Beffinger M, Schellhammer L, Vom BJ, Wollscheid B, Beule D, Paolicelli R,
Meyer U (2020). “Enzymatic Dissociation Induces Transcriptional and Proteotype Bias in Brain Cell Populations.” _Int J Mol Sci_, 21.
doi:10.3390/ijms21217944  [☞ Link](https://doi.org/10.3390/ijms21217944)

Kuhring M, Eisenberger A, Schmidt V, Kränkel N, Leistner D, Kirwan J, Beule D (2020). “Concepts and Software Package for Efficient Quality Control in
Targeted Metabolomics Studies: MeTaQuaC.” _Anal Chem_, 92, 10241-10245. doi:10.1021/acs.analchem.0c00136
 [☞ Link](https://doi.org/10.1021/acs.analchem.0c00136)

Koche R, Rodriguez-Fos E, Helmsauer K, Burkert M, MacArthur I, Maag J, Chamorro R, Munoz-Perez N, Puiggròs M, Dorado GH, Bei Y, Röefzaad C, Bardinet V,
Szymansky A, Winkler A, Thole T, Timme N, Kasack K, Fuchs S, Klironomos F, Thiessen N, Blanc E, Schmelz K, Künkele A, Hundsdörfer P, Rosswog C, Theissen
J, Beule D, Deubzer H, Sauer S, Toedling J, Fischer M, Hertwig F, Schwarz R, Eggert A, Torrents D, Schulte J, Henssen A (2020). “Extrachromosomal
circular DNA drives oncogenic genome remodeling in neuroblastoma.” _Nat Genet_, 52, 29-34. doi:10.1038/s41588-019-0547-z
 [☞ Link](https://doi.org/10.1038/s41588-019-0547-z)

Holtgrewe M, Stolpe O, Nieminen M, Mundlos S, Knaus A, Kornak U, Seelow D, Segebrecht L, Spielmann M, Fischer-Zirnsak B, Boschann F, Scholl U, Ehmke N,
Beule D (2020). “VarFish: comprehensive DNA variant analysis for diagnostics and research.” _Nucleic Acids Res_, 48, W162-W169.
doi:10.1093/nar/gkaa241  [☞ Link](https://doi.org/10.1093/nar/gkaa241)

Obermayer B, Holtgrewe M, Nieminen M, Messerschmidt C, Beule D (2020). “SCelVis: exploratory single cell data analysis on the desktop and in the cloud.”
_PeerJ_, 8, e8607. doi:10.7717/peerj.8607  [☞ Link](https://doi.org/10.7717/peerj.8607)

Nieminen M, Stolpe O, Schumann F (2020). “SODAR Core: a Django-based framework for scientific data management and analysis web apps.” _Journal of Open
Source Software_, 5(55), 1584. doi:10.21105/joss.01584  [☞ Link](http://edoc.mdc-berlin.de/19748/1/19748oa.pdf)

Akpa C, Kleo K, Oker E, Tomaszewski N, Messerschmidt C, López C, Wagener R, Oehl-Huber K, Dettmer K, Schoeler A, Lenze D, Oefner P, Beule D, Siebert R,
Capper D, Dimitrova L, Hummel M (2020). “Acquired resistance to DZNep-mediated apoptosis is associated with copy number gains of AHCY in a B-cell
lymphoma model.” _BMC Cancer_, 20, 427. doi:10.1186/s12885-020-06937-8  [☞ Link](https://doi.org/10.1186/s12885-020-06937-8)

Messerschmidt C, Obermayer B, Klinghammer K, Ochsenreither S, Treue D, Stenzinger A, Glimm H, Fröhling S, Kindler T, Brandts C, Schulze-Osthoff K,
Weichert W, Tinhofer I, Klauschen F, Keilholz U, Beule D, Rieke D (2020). “Distinct immune evasion in APOBEC-enriched, HPV-negative HNSCC.” _Int J
Cancer_, 147, 2293-2302. doi:10.1002/ijc.33123  [☞ Link](https://doi.org/10.1002/ijc.33123)

Boschann F, Fischer-Zirnsak B, Wienker T, Holtgrewe M, Seelow D, Eichhorn B, Döhnert S, Fahsold R, Horn D, Graul-Neumann L (2020). “An intronic splice
site alteration in combination with a large deletion affecting VPS13B (COH1) causes Cohen syndrome.” _Eur J Med Genet_, 63, 103973.
doi:10.1016/j.ejmg.2020.103973  [☞ Link](https://doi.org/10.1016/j.ejmg.2020.103973)

Rieke D, Klinghammer K, Obermayer B, Blanc E, Messerschmidt C, Jöhrens K, Klauschen F, Tinhofer I, Beule D, Keilholz U, Ochsenreither S (2020).
“Association of a STK11/KEAP1-Mutation Gene Expression Signature in Lung Adenocarcinoma with Immune Desertion in Squamous Cell Carcinomas and Mediation
by NFE2L2 Deregulation.” _J Clin Oncol_, 38(15_suppl), 3082-3082. doi:10.1200/JCO.2020.38.15_suppl.3082
 [☞ Link](https://doi.org/10.1200/JCO.2020.38.15_suppl.3082)

Ehmke N, Cusmano-Ozog K, Koenig R, Holtgrewe M, Nur B, Mihci E, Babcock H, Gonzaga-Jauregui C, Overton J, Xiao J, Martinez A, Muenke M, Balzer A, Jochim
J, El CN, Fischer-Zirnsak B, Huber C, Kornak U, Elsea S, Cormier-Daire V, Ferreira C (2020). “Biallelic variants in KYNU cause a multisystemic syndrome
with hand hyperphalangism.” _Bone_, 133, 115219. doi:10.1016/j.bone.2019.115219  [☞ Link](https://doi.org/10.1016/j.bone.2019.115219)

Seidel E, Walenda G, Messerschmidt C, Obermayer B, Peitzsch M, Wallace P, Bahethi R, Yoo T, Choi M, Schrade P, Bachmann S, Liebisch G, Eisenhofer G,
Beule D, Scholl U (2020). “Generation and characterization of a mitotane-resistant adrenocortical cell line.” _Endocr Connect_, 9, 122-134.
doi:10.1530/EC-19-0510  [☞ Link](https://doi.org/10.1530/EC-19-0510)

Melo U, Schöpflin R, Acuna-Hidalgo R, Mensah M, Fischer-Zirnsak B, Holtgrewe M, Klever M, Türkmen S, Heinrich V, Pluym I, Matoso E, Bernardo dSS, Louro
P, Hülsemann W, Cohen M, Dufke A, Latos-Bieleńska A, Vingron M, Kalscheuer V, Quintero-Rivera F, Spielmann M, Mundlos S (2020). “Hi-C Identifies Complex
Genomic Rearrangements and TAD-Shuffling in Developmental Diseases.” _Am J Hum Genet_, 106, 872-884. doi:10.1016/j.ajhg.2020.04.016
 [☞ Link](https://doi.org/10.1016/j.ajhg.2020.04.016)

Robinson P, Ravanmehr V, Jacobsen J, Danis D, Zhang X, Carmody L, Gargano M, Thaxton C, UNC BC, Karlebach G, Reese J, Holtgrewe M, Köhler S, McMurry J,
Haendel M, Smedley D (2020). “Interpretable Clinical Genomics with a Likelihood Ratio Paradigm.” _Am J Hum Genet_, 107, 403-417.
doi:10.1016/j.ajhg.2020.06.021  [☞ Link](https://doi.org/10.1016/j.ajhg.2020.06.021)

Blumenau S, Foddis M, Müller S, Holtgrewe M, Bentele K, Berchtold D, Beule D, Dirnagl U, Sassi C (2020). “Investigating APOE, APP-Aβ metabolism genes
and Alzheimer's disease GWAS hits in brain small vessel ischemic disease.” _Sci Rep_, 10, 7103. doi:10.1038/s41598-020-63183-5
 [☞ Link](https://doi.org/10.1038/s41598-020-63183-5)

Quade A, Thiel A, Kurth I, Holtgrewe M, Elbracht M, Beule D, Eggermann K, Scholl U, Häusler M (2020). “Paroxysmal tonic upgaze: A heterogeneous clinical
condition responsive to carbonic anhydrase inhibition.” _Eur J Paediatr Neurol_, 25, 181-186. doi:10.1016/j.ejpn.2019.11.002
 [☞ Link](https://doi.org/10.1016/j.ejpn.2019.11.002)

Thole T, Toedling J, Sprüssel A, Pfeil S, Savelyeva L, Capper D, Messerschmidt C, Beule D, Groeneveld-Krentz S, Eckert C, Gambara G, Henssen A, Finkler
S, Schulte J, Sieber A, Bluethgen N, Regenbrecht C, Künkele A, Lodrini M, Eggert A, Deubzer H (2020). “Reflection of neuroblastoma intratumor
heterogeneity in the new OHC-NB1 disease model.” _Int J Cancer_, 146, 1031-1041. doi:10.1002/ijc.32572  [☞ Link](https://doi.org/10.1002/ijc.32572)

Lamping M, Benary M, Leyvraz S, Messerschmidt C, Blanc E, Kessler T, Schütte M, Lenze D, Jöhrens K, Burock S, Klinghammer K, Ochsenreither S, Sers C,
Schäfer R, Tinhofer I, Beule D, Klauschen F, Yaspo M, Keilholz U, Rieke D (2020). “Support of a molecular tumour board by an evidence-based decision
management system for precision oncology.” _Eur J Cancer_, 127, 41-51. doi:10.1016/j.ejca.2019.12.017  [☞ Link](https://doi.org/10.1016/j.ejca.2019.12.017)

Praktiknjo S, Obermayer B, Zhu Q, Fang L, Liu H, Quinn H, Stoeckius M, Kocks C, Birchmeier W, Rajewsky N (2020). “Tracing tumorigenesis in a solid tumor
model at single-cell resolution.” _Nat Commun_, 11, 991. doi:10.1038/s41467-020-14777-0  [☞ Link](https://doi.org/10.1038/s41467-020-14777-0)

Suter A, Santos-Simarro F, Toerring P, Abad PA, Ramos-Mejia R, Heath K, Huckstadt V, Parrón-Pajares M, Mensah M, Hülsemann W, Holtgrewe M, Mundlos S,
Kornak U, Bartsch O, Ehmke N (2020). “Variable pulmonary manifestations in Chitayat syndrome: Six additional affected individuals.” _Am J Med Genet A_,
182, 2068-2076. doi:10.1002/ajmg.a.61735  [☞ Link](https://doi.org/10.1002/ajmg.a.61735)

Bockstahler M, Fischer A, Goetzke C, Neumaier H, Sauter M, Kespohl M, Müller A, Meckes C, Salbach C, Schenk M, Heuser A, Landmesser U, Weiner J, Meder
B, Lehmann L, Kratzer A, Klingel K, Katus H, Kaya Z, Beling A (2020). “Heart-Specific Immune Responses in an Animal Model of Autoimmune-Related
Myocarditis Mitigated by an Immunoproteasome Inhibitor and Genetic Ablation.” _Circulation_, 141, 1885-1902. doi:10.1161/CIRCULATIONAHA.119.043171
 [☞ Link](https://doi.org/10.1161/CIRCULATIONAHA.119.043171)

Krishnamoorthy G, Kaiser P, Abu AU, Weiner J3, Moura-Alves P, Brinkmann V, Kaufmann S (2020). “FX11 limits Mycobacterium tuberculosis growth and
potentiates bactericidal activity of isoniazid through host-directed activity.” _Dis Model Mech_, 13. doi:10.1242/dmm.041954
 [☞ Link](https://doi.org/10.1242/dmm.041954)

Tregoning J, Weiner J, Cizmeci D, Hake D, Maertzdorf J, Kaufmann S, Leroux-Roels G, Maes C, Aerssens A, Calvert A, Jones C (2020). “Pregnancy has a
minimal impact on the acute transcriptional signature to vaccination.” _NPJ Vaccines_, 5, 29. doi:10.1038/s41541-020-0177-6
 [☞ Link](https://doi.org/10.1038/s41541-020-0177-6)

Weiner J, Domaszewska T, Donkor S, Kaufmann S, Hill P, Sutherland J (2020). “Changes in Transcript, Metabolite, and Antibody Reactivity During the Early
Protective Immune Response in Humans to Mycobacterium tuberculosis Infection.” _Clin Infect Dis_, 71, 30-40. doi:10.1093/cid/ciz785
 [☞ Link](https://doi.org/10.1093/cid/ciz785)

Win Z, Weiner J, Listanco A, Patel N, Sharma R, Greenwood A, Maertzdorf J, Mollenkopf H, Pizzoferro K, Cole T, Bodinham C, Kaufmann S, Denoel P, Del GG,
Lewis D (2020). “Systematic Evaluation of Kinetics and Distribution of Muscle and Lymph Node Activation Measured by (18)F-FDG- and (11)C-PBR28-PET/CT
Imaging, and Whole Blood and Muscle Transcriptomics After Immunization of Healthy Humans With Adjuvanted and Unadjuvanted Vaccines.” _Front Immunol_,
11, 613496. doi:10.3389/fimmu.2020.613496  [☞ Link](https://doi.org/10.3389/fimmu.2020.613496)

Mochel F, Rastetter A, Ceulemans B, Platzer K, Yang S, Shinde D, Helbig K, Lopergolo D, Mari F, Renieri A, Benetti E, Canitano R, Waisfisz Q, Plomp A,
Huisman S, Wilson G, Cathey S, Louie R, Gaudio D, Waggoner D, Kacker S, Nugent K, Roeder E, Bruel A, Thevenon J, Ehmke N, Horn D, Holtgrewe M, Kaiser F,
Kamphausen S, Abou JR, Weckhuysen S, Dalle C, Depienne C (2020). “Variants in the SK2 channel gene (KCNN2) lead to dominant neurodevelopmental movement
disorders.” _Brain_, 143(12), 3564-3573. doi:10.1093/brain/awaa346  [☞ Link](https://doi.org/10.1093/brain/awaa346)

Foddis M, Winek K, Bentele K, Mueller S, Blumenau S, Reichhart NN, Crespo-Garcia S, Harnett D, Ivanov A, Meisel A, Joussen A, Strauss O, Beule D,
Dirnagl U, Sassi C (2020). “An exploratory investigation of brain collateral circulation plasticity after cerebral ischemia in two experimental C57BL/6
mouse models.” _J Cereb Blood Flow Metab_, 40, 276-287. doi:10.1177/0271678X19827251  [☞ Link](https://doi.org/10.1177/0271678X19827251)

Quade A, Thiel A, Kurth I, Holtgrewe M, Elbracht M, Beule D, Eggermann K, Scholl U, Häusler M (2020). “Paroxysmal tonic upgaze: A heterogeneous clinical
condition responsive to carbonic anhydrase inhibition.” _Eur J Paediatr Neurol_, 25, 181-186. doi:10.1016/j.ejpn.2019.11.002
 [☞ Link](https://doi.org/10.1016/j.ejpn.2019.11.002)


### 2019

Kircher, M., Xiong, C., Martin, B. et al.
"Saturation mutagenesis of twenty disease-associated regulatory elements at single base-pair resolution."
*Nat Commun*
10, 3583 (2019).
[doi: 10.1038/s41467-019-11526-w](https://doi.org/10.1038/s41467-019-11526-w)

Stefanovski L, Triebkorn P, Spiegler A, Diaz-Cortes M-A, Solodkin A, Jirsa V, McIntosh RA and Ritter P (2019).
"Linking Molecular Pathways and Large-Scale Computational Modeling to Assess Candidate Disease Mechanisms and Pharmacodynamics in Alzheimer's Disease."
*Front. Comput. Neurosci.*. 13:54.
[doi: 10.3389/fncom.2019.00054](https://doi.org/10.3389/fncom.2019.00054)

Boeddrich A., Babila J.T., Wiglenda T., Diez L., Jacob M., Nietfeld W., Huska M.R., Haenig C., Groenke N., Buntru A., Blanc E., Meier J.C., Vannoni E., Erck C., Friedrich B., Martens H., Neuendorf N., Schnoegl S., Wolfer DP., Loos M., Beule D., Andrade-Navarro M.A., Wanker E.E. (2019).
"The Anti-amyloid Compound DO1 Decreases Plaque Pathology and Neuroinflammation-Related Expression Changes in 5xFAD Transgenic Mice."
*Cell Chem Biol.*
2019 Jan 17;26(1):109-120.e7.
[doi: 10.1016/j.chembiol.2018.10.013](https://doi.org/10.1016/j.chembiol.2018.10.013).

Fountain M.D., Oleson, D.S., Rech. M.E., Segebrecht, L., Hunter, J.V., McCarthy, J.M., Lupo, P.J., Holtgrewe, M., Mora, R., Rosenfeld, J.A., Isidor, B., Le Caignec, C., Saenz, M.S., Pedersen, R.C., Morgen, T.M., Pfotenhauer, J.P., Xia, F., Bi, W., Kang, S.-H.L., Patel, A., Krantz, I.D., Raible, S.E., Smith, W.E., Cristian, I., Tori, E., Juusola, J., Millan, F., Wentzensen, I.M., Person, R.E., Küry, S., Bézieau, S., Uguen, K., Férec, C., Munnich, A., van Haelst, M., Lichtenbelt, K.D., van Gassen, K., Hagelstrom, T., Chawla, A., Perry, D.L., Taft, R.J., Jones, M., Masser-Frye, D., Dyment, D., Venkateswaran, S., Li, C., Escobar, L,.F., Horn, D., Spillmann, R.C., Peña, L., Wierzba, J., Strom, T.M. Parent, I. Kaiser, F.J., Ehmke, N., Schaaf, C.P. (2019).
"Pathogenic variants in USP7 cause a neurodevelopmental disorder with speech delays, altered behavior, and neurologic anomalies."
*Genet. Med.*
2019 Jan 25.
[doi: 10.1038/s41436-019-0433-1](https://doi.org/10.1038/s41436-019-0433-1)

Holtgrewe,M., Messerschmidt,C., Nieminen,M. and Beule,D. (2019) DigestiFlow: from BCL to FASTQ with ease. Bioinformatics, 10.1093/bioinformatics/btz850.

Käfer S., Paraskevopoulou S., Zirkel F., Wieseke N., Donath A., Petersen M., Jones T.C., Liu S., Zhou X., Middendorf M., Junglen S., Misof B., Drosten C. (2019). 
"Re-assessing the diversity of negative strand RNA viruses in insects."
*PLOS Pathogens*
2019 Dec 12. 
[doi: 10.1371/journal.ppat.1008224](https://doi.org/10.1371/journal.ppat.1008224)

Kühnisch,J., Herbst,C., Al‐Wakeel‐Marquard,N., Dartsch,J., Holtgrewe,M., Baban,A., Mearini,G., Hardt,J., Kolokotronis,K., Gerull,B., et al. (2019) Targeted panel sequencing in pediatric primary cardiomyopathy supports a critical role of TNNI3. Clin Genet, 96, 549–559.
https://doi.org/10.1111/cge.13645

Marklewitz M., Dutari L.C., Paraskevopoulou S., Page R.A., Loaiza J.R., Junglen S. (2019). 
"Diverse novel phleboviruses in sandflies from the Panama Canal area, Central Panama."
*Journal of General Virology*
2019 May 3.
[doi: 10.1099/jgv.0.001260](https://doi.org/10.1099/jgv.0.001260)

Quade,A., Thiel,A., Kurth,I., Holtgrewe,M., Elbracht,M., Beule,D., Eggermann,K., Scholl,U.I. and Häusler,M. (2019) Paroxysmal tonic upgaze: A heterogeneous clinical condition responsive to carbonic anhydrase inhibition. European Journal of Paediatric Neurology, [10.1016/j.ejpn.2019.11.002](https://doi.org/10.1016/j.ejpn.2019.11.002).

Holtgrewe M, Messerschmidt C, Nieminen M, Beule D (2019). “Digestiflow: From BCL to FASTQ with Ease.” _Bioinformatics_.

Schumann F, Blanc E, Messerschmidt C, Blankenstein T, Busse A, Beule D (2019). “SigsPack, a package for cancer mutational signatures.” _BMC
Bioinformatics_, 20, 450. doi:10.1186/s12859-019-3043-7  [☞ Link](https://doi.org/10.1186/s12859-019-3043-7)

Blanc E, Holtgrewe M, Dhamodaran A, Messerschmidt C, Willimsky G, Blankenstein T, Beule D (2019). “Identification and ranking of recurrent neo-epitopes
in cancer.” _BMC Med Genomics_, 12, 171. doi:10.1186/s12920-019-0611-7  [☞ Link](https://doi.org/10.1186/s12920-019-0611-7)

Boeddrich A, Babila J, Wiglenda T, Diez L, Jacob M, Nietfeld W, Huska M, Haenig C, Groenke N, Buntru A, Blanc E, Meier J, Vannoni E, Erck C, Friedrich
B, Martens H, Neuendorf N, Schnoegl S, Wolfer D, Loos M, Beule D, Andrade-Navarro M, Wanker E (2019). “The Anti-amyloid Compound DO1 Decreases Plaque
Pathology and Neuroinflammation-Related Expression Changes in 5xFAD Transgenic Mice.” _Cell Chem Biol_, 26, 109-120.e7.
doi:10.1016/j.chembiol.2018.10.013  [☞ Link](https://doi.org/10.1016/j.chembiol.2018.10.013)

Borgsmüller N, Gloaguen Y, Opialla T, Blanc E, Sicard E, Royer A, Le BB, Durand S, Migné C, Pétéra M, Pujos-Guillot E, Giacomoni F, Guitton Y, Beule D,
Kirwan J (2019). “WiPP: Workflow for Improved Peak Picking for Gas Chromatography-Mass Spectrometry (GC-MS) Data.” _Metabolites_, 9.
doi:10.3390/metabo9090171  [☞ Link](https://doi.org/10.3390/metabo9090171)

Buonfiglioli A, Efe I, Guneykaya D, Ivanov A, Huang Y, Orlowski E, Krüger C, Deisz R, Markovic D, Flüh C, Newman A, Schneider U, Beule D, Wolf S, Dzaye
O, Gutmann D, Semtner M, Kettenmann H, Lehnardt S (2019). “let-7 MicroRNAs Regulate Microglial Function and Suppress Glioma Growth through Toll-Like
Receptor 7.” _Cell Rep_, 29, 3460-3471.e7. doi:10.1016/j.celrep.2019.11.029  [☞ Link](https://doi.org/10.1016/j.celrep.2019.11.029)

Kühnisch J, Herbst C, Al-Wakeel-Marquard N, Dartsch J, Holtgrewe M, Baban A, Mearini G, Hardt J, Kolokotronis K, Gerull B, Carrier L, Beule D, Schubert
S, Messroghli D, Degener F, Berger F, Klaassen S (2019). “Targeted panel sequencing in pediatric primary cardiomyopathy supports a critical role of
TNNI3.” _Clin Genet_, 96, 549-559. doi:10.1111/cge.13645  [☞ Link](https://doi.org/10.1111/cge.13645)

Kuhring M, Nieminen M, Kirwan J, Beule D, Holtgrewe M (2019). “AltamISA: A Python API for ISA-Tab Files.” _JOSS_, 4, 1610. doi:10.21105/joss.01610


Lamping M, Rieke D, Klauschen F, Jöhrens K, Anagnostopoulos I, Lenze D, Tinhofer I, Benary M, Ochsenreither S, Klinghammer K, Burock S, Jann H, Stüven
A, Ditzen D, Beule D, Messerschmidt C, Blanc E, Schäfer R, Keilholz U (2019). “Clinical Impact of Comprehensive versus Targeted Genomic Analysis for
Precision Oncology.” _J Clin Oncol_, 37(15_suppl), e13033-e13033. doi:10.1200/JCO.2019.37.15_suppl.e13033
 [☞ Link](https://doi.org/10.1200/JCO.2019.37.15_suppl.e13033)

Herda S, Heimann A, Althoff S, Ruß J, Bullinger L, Beule D, Obermayer B, Na I (2019). “Long-Term T Cell Expansion Results in Increased Numbers of
Central Memory T Cells with Sustained Functional Properties for Adoptive T Cell Therapy.” _Blood_. doi:10.1182/blood-2019-124736
 [☞ Link](https://www.sciencedirect.com/science/article/pii/S0006497118598608)

van Heesch S, Witte F, Schneider-Lunitz V, Schulz J, Adami E, Faber A, Kirchner M, Maatz H, Blachut S, Sandmann C, Kanda M, Worth C, Schafer S,
Calviello L, Merriott R, Patone G, Hummel O, Wyler E, Obermayer B, Mücke M, Lindberg E, Trnka F, Memczak S, Schilling M, Felkin L, Barton P, Quaife N,
Vanezis K, Diecke S, Mukai M, Mah N, Oh S, Kurtz A, Schramm C, Schwinge D, Sebode M, Harakalova M, Asselbergs F, Vink A, de WR, Viswanathan S, Widjaja
A, Gärtner-Rommel A, Milting H, Dos RC, Knosalla C, Mertins P, Landthaler M, Vingron M, Linke W, Seidman J, Seidman C, Rajewsky N, Ohler U, Cook S,
Hubner N (2019). “The Translational Landscape of the Human Heart.” _Cell_, 178, 242-260.e29. doi:10.1016/j.cell.2019.05.010
 [☞ Link](https://doi.org/10.1016/j.cell.2019.05.010)




### 2018

Blanc, E., Holtgrewe, M., Dhamodaran, A., Messerschmidt, C., Willimsky, G., Blankenstein, T., Beule, D. (2018).
"Identification and Ranking of Recurrent Neo-Epitopes in Cancer".
*bioRxiv*.
2018/389437, 2018.
[doi: 10.1101/389437](https://doi.org/10.1101/389437)

Brandt, R., Uhlitz, F., Riemer, P., Giesecke, C., Schulze, S., El-Shimy, I.A., Fauler, B., Mielke, T., Mages, N., Herrmann, B.G., Sers, C., Blüthgen, N., Morkel, M. (2018).
"Cell type-dependent differential activation of ERK by oncogenic KRAS or BRAF in the mouse intestinal epithelium".
*bioRxiv*.
2018/340844.
[doi: 10.1101/340844](https://dx.doi.org/10.1101/340844).

Holtgrewe, M., Knaus, A., Hildebrand, G., Pantel, J.-T., Rodriguesz de los Santos, M., Neveling, K., Goldmann, J., Schubach, M., Jäger, M., Couterier, M., Mundlos, S., Beule, D., Sperling, K., Krawitz, P. (2018).
"Multisite de novo mutations in human offspring after paternal exposure to ionizing radiation",
*Nature Scientific Reports*. 2018 Oct 2;8(1):14611.
[doi: 10.1038/s41598-018-33066-x](https://doi.org/10.1038/s41598-018-33066-x).

Kircher M., Xiong C., Martin B, Schubach M, Inoue F, Bell R.JA., Costello J.F., Shendure J., Ahituv N. (2018).
"Saturation mutagenesis of disease-associated regulatory elements." bioRxiv (2018): 505362. [doi: 10.1101/505362](https://doi.org/10.1101/505362)

PCAWG Transcriptome Core Group, Calabrese, C., Davidson, N.R., Fonseca1, N.A., He, Y., Kahles, A., Lehmann, K.-V., Liu, F., Shiraishi, Y., Soulette, C.M., Urban, L., Demircioğlu, D., Greger, L., Li, S., Liu, D., Perry, M.D., Xiang, L., Zhang, F., Zhang, J., Bailey, P., Erkek, S., Hoadley, K.A., Hou, Y., Kilpinen, H., Korbel, J.O., Marin, M.G., Markowski, J., Nandi11, T., Pan-Hammarström, Q., Pedamallu, C.S., Siebert, R., Stark, S.G., Su, H., Tan, P., Waszak, S.M., Yung, C., Zhu, S., PCAWG Transcriptome Working Group, Awadalla, P., Creighton, C.J., Meyerson, M., Ouellette, B.F.F., Wu, K., Yang, H., ICGC/TCGA Pan-Cancer Analysis of Whole Genomes Network, Brazma1, A., Brooks, A.N., Göke, J., Rätsch, G., Schwarz, R.F., Stegle, O., Zhang, Z. (2018).
"Genomic basis for RNA alterations revealed by whole-genome analyses of 27 cancer types".
*bioRxiv*.
2018/183889.
[doi: 10.1101/183889 ](https://doi.org/10.1101/183889)

Guneykaya D., Ivanov A., Hernandez D.P., Haage V., Wojtas B., Meyer N., Maricos M., Jordan P., Buonfiglioli A., Gielniewski B., Ochocka N., Cömert, C., Friedrich, C., Artiles, L. S., Kaminska, B., Mertins, P., Beule, D., Kettenmann, H. (2018).
"Transcriptional and translational differences of microglia from male and female brains",
*Cell reports*. 2018 Sep 4;24(10):2773-83.
[doi: 10.1016/j.celrep.2018.08.001](https://doi.org/10.1016/j.celrep.2018.08.001).

Rentzsch P, Witten D, Cooper GM, Shendure J, Kircher M. (2018).
"CADD: predicting the deleteriousness of variants throughout the human genome",
*Nucleic Acids Res*. 2018 Oct 29. [doi: 10.1093/nar/gky1016](https://doi.org/10.1093/nar/gky1016).

Salatzki J., Foryst-Ludwig A., Bentele K., Blumrich A., Smeir E., Ban Z., Brix S., Grune J., Beyhoff N., Klopfleisch R., Dunst S., Surma, M.A., Klose, C., Rothe, M., Heinzel, F.R., Krannich, A., Kershaw, E.E., Beule, D., Schulze, P.C., Marx, N., Kintscher, U. (2018).
"Adipose tissue ATGL modifies the cardiac lipidome in pressure-overload-induced left ventricular failure",
*PLoS genetics*. 2018 Jan 10;14(1):e1007171.
[doi: 10.1371/journal.pgen.100717](https://doi.org/10.1371/journal.pgen.1007171).

Schubach M., Re M., Robinson P.N., Valentini G. (2017)
"Imbalance-aware machine learning for predicting rare and common disease-associated non-coding variants",
*Scientific reports* 
7:1, 2959.
[doi: 10.1038/s41598-017-03011-5](https://doi.org/10.1038/s41598-017-03011-5).

Schubert M., Klinge, B., Klünemann M., Sieber A., Uhlitz F., Sauer S., Garnett M., Blüthgen N., Saez-Rodriguez J. (2018).
"Perturbation-response genes reveal signaling footprints in cancer gene expression". 
*Nature Communications*.
9: 20, 2018.
[doi: 10.1038/s41467-017-02391-6](https://doi.org/10.1038/s41467-017-02391-6)

### 2017

Euskirchen, P., Bielle, F., Labreche, K., Kloosterman, W.P., Rosenberg, S., Daniau, M., Schmitt, C., Masliah-Planchon, J., Bourdeaut, F., Dehais, C., et al. (2017). Same-day genomic and epigenomic diagnosis of brain tumors using real-time nanopore sequencing. Acta Neuropathol 1–13. [doi: 10.1007/s00401-017-1743-5](https://doi.org/10.1007/s00401-017-1743-5)

Euskirchen, P., Radke, J., Schmidt, M.S., Heuling, E.S., Kadikowski, E., Maricos, M., Knab, F., Grittner, U., Zerbe, N., Czabanka, M., et al. (2017). Cellular heterogeneity contributes to subtype-specific expression of ZEB1 in human glioblastoma. PLOS ONE 12, e0185376. [doi: 10.1371/journal.pone.0185376](https://doi.org/10.1371/journal.pone.0185376 )

Mattei D., Ivanov A., Ferrai C., Jordan P., Guneykaya D., Buonfiglioli A., Schaafsma W., Przanowski P., Deuther-Conrad W., Brust P., Hesse S., Patt, M., Sabri, O., Ross, T.L., Eggen, B.J.L., Boddeke E.W.G.M., Kaminska, B., Beule, D., Pombo, A., Kettenmann, H., Wolf, S.A. (2017).
"Maternal immune activation results in complex microglial transcriptome signature in the adult offspring that is reversed by minocycline treatment." *Translational psychiatry*. 2017 May;7(5):e1120.
[doi: 10.1038/tp.2017.80](https://doi.org/10.1038/tp.2017.80).

Mamlouk, S., Childs, L. H., Aust, D., Heim, D., Melching, F., Oliveira, C., Wolf, T., Durek, P., Schumacher, D., Bläker, H., von Winterfeld, M., Gastl, B., Möhr, K., Menne, A., Zeugner, S., Redmer, T., Lenze, D., Tierling, S., Möbs, M., Weichert, W., Folprecht, G., Blanc, E., Beule, D., Schäfer, R., Morkel, M., Klauschen, F., Leser, U. and Sers, C. (2017).
"DNA copy number changes define spatial patterns of heterogeneity in colorectal cancer",
*Nature Communications*. 2017; 8, p. 14093.
[doi: 10.1038/ncomms14093](http://doi.org/10.1038/ncomms14093).

Messerschmidt, C., Holtgrewe, M. and Beule, D. (2017).
"HLA-MA: simple yet powerful matching of samples using HLA typing results".
*Bioinformatics*. 28, pp. 2592–2599.
[doi: 10.1093/bioinformatics/btx132](https://doi.org/10.1101/066548).

Kammertoens, T., Friese, C., Arina, A., Idel, C., Briesemeister, D., Rothe, M., Ivanov, A., Szymborska, A., Patone, G., Kunz, S., Sommermeyer, D., Engels, B., Leisegang, M., Textor, A., Fehling, H. J., Fruttiger, M., Lohoff, M., Herrmann, A., Yu, H., Weichselbaum, R., Uckert, W., Hübner, N., Gerhardt, H., Beule, D., Schreiber, H. and Blankenstein, T. (2017).
"Tumour ischaemia by interferon-γ resembles physiological blood vessel regression".
*Nature*. 545(7652), pp. 98–102.
[doi: 10.1038/nature22311](https://doi.org/10.1038/nature22311).

Schulze Heuling, E., Knab, F., Radke, J., Eskilsson, E., Martinez-Ledesma, E., Koch, A., Czabanka, M., Dieterich, C., Verhaak, R.G., Harms, C., et al. (2017). Prognostic Relevance of Tumor Purity and Interaction with MGMT Methylation in Glioblastoma. Mol. Cancer Res. 15, 532–540. [doi: 10.1158/1541-7786.MCR-16-0322 ](https://doi.org/10.1158/1541-7786.MCR-16-0322 )

Yaakov, G., Lerner, D., Bentele, K., Steinberger, J., Barkai, N., Bigger, J., Maisonneuve, E., Gerdes, K., Lewis, K., Dhar, N., McKinney, J. D., Gefen, O., Balaban, N. Q., Jayaraman, R., Balaban, N. Q., Merrin, J., Chait, R., Kowalik, L., Leibler, S., Balaban, N. Q., Allison, K. R., Brynildsen, M. P., Collins, J. J., Nathan, C., Lewis, K., Glickman, M. S., Sawyers, Knoechel, B., Welch, A. Z., Gibney, P. A., Botstein, D., Koshland, D. E., Levy, S. F., Ziv, N., Siegal, M. L., Stewart-Ornstein, J., Weissman, J. S., El-Samad, H., Gasch, A. P., Weinert, T., Hartwell, L., Weinert, T. A., Hartwell, L. H., Lisby, M., Rothstein, R., Mortensen, U. H., Lisby, M., Mortensen, U. H., Rothstein, R., Domkin, V., Thelander, L., Chabes, A., Hendry, J. A., Tan, G., Ou, J., Boone, C., Brown, G. W., Berry, D. B., Gasch, A. P., Lynch, M., Nishant, K. T., Serero, A., Jubin, C., Loeillet, S., Legoix-Ne, P., Nicolas, A. G., Huh, W. K., Janke, C., Lee, S. E., Blecher-Gonen, R., Martin, M., Cherry, J. M., McKenna, A., DePristo, M. A., Lawrence, M., Obenchain, V., Ye, K., Schulz, M. H., Long, Q., Apweiler, R., Ning, Z., Layer, R. M., Chiang, C., Quinlan, A. R., Hall, I. M., Faust, G. G., Hall, I. M., Boeva, V., Boeva, V., Li, H., Koren, A., Soifer, I. and Barkai, N. (2017).
"Coupling phenotypic persistence to DNA damage increases genetic diversity in severe stress".
*Nature Ecology & Evolution*. 1(1), pp. 497–500.
[doi: 10.1038/s41559-016-0016](https://doi.org/10.1038/s41559-016-0016).

Uhlitz, F., Sieber, A., Wyler, E., Fritsche-Guenther, R., Meisig, J., Landthaler, M., Klinger, B., Blüthgen, N. (2017).
"An immediate-late gene expression module decodes ERK signal duration".
*Molecular Systems Biology*.
13: 928, 2017.
[doi: 10.15252/msb.20177554](https://doi.org/10.15252/msb.20177554).

## Theses

### 2019

Schumann F. (2019).
"Establishing a pipeline for stable mutational signature detection and evaluation of variant filter effects". *Freie Universität Berlin*.
Bachelor Thesis, Bioinformatics.

### 2018

Borgsmüller N. (2018).
"Optimization of data processing in GC-MS metabolomics",
*Technische Universität Berlin*.
Master Thesis, Biotechnology.

Kuchenbecker, S.-L. (2018).
"Analysis of Antigen Receptor Repertoires Captured by High Throughput Sequencing".
*Freie Universität Universität Berlin*.
PhD Thesis, Dr. rer. nat.
[URN:NBN: urn:nbn:de:kobv:188-refubium-22171-8](https://nbn-resolving.de/urn:nbn:de:kobv:188-refubium-22171-8)

Schubach M. (2018).
"Learning the Non-Coding Genome",
*Freie Universität Universität Berlin*.
PhD Thesis, Dr. rer. nat.
[URN:NBN: urn:nbn:de:kobv:188-refubium-23332-7](https://nbn-resolving.de/urn:nbn:de:kobv:188-refubium-23332-7)

## Posters

### 2018

Roskosch, S., Haldórsson B., Kehr, B. (2018).
"PopDel: Population-Scale Detection of Genomic Deletions"
*ECCB 2018.*
Poster.

White T., Kehr B. (2018).
"Comprehensive extraction of structural variations from long-read DNA sequences"
*WABI 2018.*
Poster.

### 2017

Schubach M., Re R., Robinson P.N., Valentini G. (2017).
"Variant relevance prediction in extremely imbalanced training sets"
*ISMB/ECCB 2017.*
Poster.

White T., Kehr B. (2017).
"Improving long-read mapping with simple lossy sequence transforms"
*ISMB/ECCB 2017.*
Poster.

