# Publication List

The BIH Cluster is a valuable resource.
It has been used to support the publications listed below.

- Please add your publications here.
- Acknowledge usage of the cluster in your manuscript as *"Computation has been performed on the HPC for Research cluster of the Berlin Institute of Health"*.

## Articles & Preprints

### 2024

Hollunder, B., Ostrem, J.L., Sahin, I.A., Rajamani, N., Oxenford, S., Butenko, K., Neudorfer, C., Reinhardt, P., Zvarova, P., Polosan, M., Akram, H., Vissani, M., Zhang, C., Sun, B., Navratil, P., Reich, M.M., Volkmann, J., Yeh, F.-C., Baldermann, J.C., Dembek, T.A., Visser-Vandewalle, V., Alho, E.J.L., Franceschini, P.R., Nanda, P., Finke, C., Kühn, A.A., Dougherty, D.D., Richardson, R.M., Bergman, H., DeLong, M.R., Mazzoni, A., Romito, L.M., Tyagi, H., Zrinzo, L., Joyce, E.M., Chabardes, S., Starr, P.A., Li, N., Horn, A., 2024. Mapping dysfunctional circuits in the frontal cortex using deep brain stimulation. Nat. Neurosci. 1–14. [doi: 10.1038/s41593-024-01570-1](https://doi.org/10.1038/s41593-024-01570-1)

### 2022

Kossen T, Hirzel MA, Madai VI, Boenisch F, Hennemuth A, Hildebrand K, Pokutta S, Sharma K, Hilbert A, Sobesky J, Galinovic I, Khalil AA, Fiebach JB and Frey D. Toward Sharing Brain Images: Differentially Private TOF-MRA Images With Segmentation Labels Using Generative Adversarial Networks. *Frontiers in
Artificial Intelligence*. 5 (2022). issn: 2624-8212. [doi: 10.3389/frai.2022.813842](https://doi.org/10.3389/frai.2022.813842)

### 2021

Li, N., Hollunder, B., Baldermann, J. C., Kibleur, A., Treu, S., Akram, H., Al-Fatly, B., Strange, B. A., Barcia, J. A., Zrinzo, L., Joyce, E. M., Chabardes, S., Visser-Vandewalle, V., Polosan, M., Kuhn, J., Kühn, A. A., & Horn, A. (2021). A Unified Functional Network Target for Deep Brain Stimulation in Obsessive-Compulsive Disorder. Biological Psychiatry. [doi: 10.1016/j.biopsych.2021.04.006](https://doi.org/10.1016/j.biopsych.2021.04.006)

Bressem KK, Vahldiek JL, Adams L, Niehues SM, Haibel H, Rodriguez VR, Torgutalp M, Protopopov M, Proft F, Rademacher J, Sieper J, Rudwaleit M, Hamm B, Makowski MR, Hermann KG, Poddubnyy D. 
Deep learning for detection of radiographic sacroiliitis: achieving expert-level performance. 
Arthritis Res Ther. 2021 Apr 8;23(1):106. [doi: 10.1186/s13075-021-02484-0](https://doi.org/10.1186/s13075-021-02484-0)

Kossen T, Subramaniam P, Madai VI, Hennemuth A, Hildebrand K, Hilbert A, Sobesky J, Livne M, Galinovic I, Khalil AA, Fiebach JB, Frey D. 
Synthesizing anonymized and labeled TOF-MRA patches for brain vessel segmentation using generative adversarial networks. 
*Computers in Biology and Medicine*. 2021 Apr 131,104254. [doi: 10.1016/j.compbiomed.2021.104254](https://doi.org/10.1016/j.compbiomed.2021.104254)

Paraskevopoulou S., Käfer S., Zirkel F., Donath A., Petersen M., Liu S., Zhou X., Drosten C., Misof B., Junglen S. (2021). 
"Viromics of extant insect orders unveil the evolution of the flavi-like superfamily."
*Virus Evolution*
2021 Mar 30. 
[doi: 10.1093/ve/veab030](https://doi.org/10.1093/ve/veab030)

Thomas Krannich, W Timothy J White, Sebastian Niehus, Guillaume Holley, Bjarni V Halldórsson, Birte Kehr, Population-scale detection of non-reference sequence variants using colored de Bruijn graphs, Bioinformatics, 2021, btab749, [doi: 10.1093/bioinformatics/btab749](https://doi.org/10.1093/bioinformatics/btab749)

Julia Markowski, Rieke Kempfer, Alexander Kukalev, Ibai Irastorza-Azcarate, Gesa Loof, Birte Kehr, Ana Pombo, Sven Rahmann, Roland F Schwarz, GAMIBHEAR: whole-genome haplotype reconstruction from Genome Architecture Mapping data, Bioinformatics, Volume 37, Issue 19, 1 October 2021, Pages 3128–3135. [doi: 10.1093/bioinformatics/btab238](https://doi.org/10.1093/bioinformatics/btab238)

### 2020

Krützfeldt LM, Schubach M, Kircher M. The impact of different negative training data on regulatory sequence predictions. PLoS One. 2020 Dec 1;15(12):e0237412. [doi: 10.1371/journal.pone.0237412](https://doi.org/10.1371/journal.pone.0237412).

Klotz-Noack K, Klinger B, Rivera M, Bublitz N, Uhlitz F, Riemer P, Lüthen M, Sell T, Kasack K, Gastl B, Ispasanie SSS, Simon T, Janssen N, Schwab M, Zuber J, Horst D, Blüthgen N, Schäfer R, Morkel M, Sers C. SFPQ Depletion Is Synthetically Lethal with BRAFV600E in Colorectal Cancer Cells. Cell Rep. 2020 Sep 22;32(12):108184. [doi: 10.1016/j.celrep.2020.108184](https://doi.org/10.1016/j.celrep.2020.108184).

Kleinert, P., Martin, B., & Kircher, M. (2020).
"HemoMIPs—Automated analysis and result reporting pipeline for targeted sequencing data."
*PLOS Computational Biology*,
16(6), e1007956.
[doi:  10.1371/journal.pcbi.1007956](https://doi.org/10.1371/journal.pcbi.1007956)

Ehmke, N.; Cusmano-Ozog, K.; Koenig, R.; Holtgrewe, M.; Nur, B.; Mihci, E.; Babcock, H.; Gonzaga-Jauregui, C.; Overton, J. D.; Xiao, J.; et al. Biallelic Variants in KYNU Cause a Multisystemic Syndrome with Hand Hyperphalangism. *Bone* 2020, 115219.
[doi: 10.1016/j.bone.2019.115219](https://doi.org/10.1016/j.bone.2019.115219).

Niehus, S.; Jónsson, H.; Schönberger, J.; Björnsson, E.; Beyter, D.; Eggertsson, H.P.; Sulem, P.; Stefánsson, K.; Halldórsson, B.V.; Kehr, B. PopDel identifies medium-size deletions jointly in tens of thousands of genomes.
*bioRxiv* 2020, 10.1101/740225
[doi: 10.1101/740225](https://doi.org/10.1101/740225)

Gordon, M. G., Inoue, F., Martin, B., Schubach, M., Agarwal, V., Whalen, S., ... & Kreimer, A. (2020).
"lentiMPRA and MPRAflow for high-throughput functional characterization of gene regulatory elements."
*Nature Protocols*, 15(8), 2387-2412.
[doi: 10.1038/s41596-020-0333-5](https://doi.org/10.1038/s41596-020-0333-5)

Paraskevopoulou S., Pirzer F., Goldmann N., Schmid J., Corman V.M., Gottula L.T.,Schroeder S., Rasche A., Muth D., Drexler J.F., Heni A.C., Eibner G.J., Page R.A., Jones T.C., MüllerM.A., Sommer S., Glebe D., and Drosten C. (2020).
"Mammalian deltavirus without hepadnavirus coinfection in the neotropical rodent *Proechimys semispinosus*."
*Proceedings of the National Academy of Sciences*
2020 Jul 28;117(30):17977-17983.
[doi: 10.1073/pnas.2006750117](https://doi.org/10.1073/pnas.2006750117).

### 2019

Kircher, M., Xiong, C., Martin, B. et al.
"Saturation mutagenesis of twenty disease-associated regulatory elements at single base-pair resolution."
*Nat Commun*
10, 3583 (2019).
[doi: 10.1038/s41467-019-11526-w](https://doi.org/10.1038/s41467-019-11526-w)

Stefanovski L, Triebkorn P, Spiegler A, Diaz-Cortes M-A, Solodkin A, Jirsa V, McIntosh RA and Ritter P (2019).
"Linking Molecular Pathways and Large-Scale Computational Modeling to Assess Candidate Disease Mechanisms and Pharmacodynamics in Alzheimer's Disease."
*Front. Comput. Neurosci.*. 13:54.
[doi: 10.3389/fncom.2019.00054](https://doi.org/10.3389/fncom.2019.00054)

Boeddrich A., Babila J.T., Wiglenda T., Diez L., Jacob M., Nietfeld W., Huska M.R., Haenig C., Groenke N., Buntru A., Blanc E., Meier J.C., Vannoni E., Erck C., Friedrich B., Martens H., Neuendorf N., Schnoegl S., Wolfer DP., Loos M., Beule D., Andrade-Navarro M.A., Wanker E.E. (2019).
"The Anti-amyloid Compound DO1 Decreases Plaque Pathology and Neuroinflammation-Related Expression Changes in 5xFAD Transgenic Mice."
*Cell Chem Biol.*
2019 Jan 17;26(1):109-120.e7.
[doi: 10.1016/j.chembiol.2018.10.013](https://doi.org/10.1016/j.chembiol.2018.10.013).

Fountain M.D., Oleson, D.S., Rech. M.E., Segebrecht, L., Hunter, J.V., McCarthy, J.M., Lupo, P.J., Holtgrewe, M., Mora, R., Rosenfeld, J.A., Isidor, B., Le Caignec, C., Saenz, M.S., Pedersen, R.C., Morgen, T.M., Pfotenhauer, J.P., Xia, F., Bi, W., Kang, S.-H.L., Patel, A., Krantz, I.D., Raible, S.E., Smith, W.E., Cristian, I., Tori, E., Juusola, J., Millan, F., Wentzensen, I.M., Person, R.E., Küry, S., Bézieau, S., Uguen, K., Férec, C., Munnich, A., van Haelst, M., Lichtenbelt, K.D., van Gassen, K., Hagelstrom, T., Chawla, A., Perry, D.L., Taft, R.J., Jones, M., Masser-Frye, D., Dyment, D., Venkateswaran, S., Li, C., Escobar, L,.F., Horn, D., Spillmann, R.C., Peña, L., Wierzba, J., Strom, T.M. Parent, I. Kaiser, F.J., Ehmke, N., Schaaf, C.P. (2019).
"Pathogenic variants in USP7 cause a neurodevelopmental disorder with speech delays, altered behavior, and neurologic anomalies."
*Genet. Med.*
2019 Jan 25.
[doi: 10.1038/s41436-019-0433-1](https://doi.org/10.1038/s41436-019-0433-1)

Holtgrewe,M., Messerschmidt,C., Nieminen,M. and Beule,D. (2019) DigestiFlow: from BCL to FASTQ with ease. Bioinformatics, 10.1093/bioinformatics/btz850.

Käfer S., Paraskevopoulou S., Zirkel F., Wieseke N., Donath A., Petersen M., Jones T.C., Liu S., Zhou X., Middendorf M., Junglen S., Misof B., Drosten C. (2019). 
"Re-assessing the diversity of negative strand RNA viruses in insects."
*PLOS Pathogens*
2019 Dec 12. 
[doi: 10.1371/journal.ppat.1008224](https://doi.org/10.1371/journal.ppat.1008224)

Kühnisch,J., Herbst,C., Al‐Wakeel‐Marquard,N., Dartsch,J., Holtgrewe,M., Baban,A., Mearini,G., Hardt,J., Kolokotronis,K., Gerull,B., et al. (2019) Targeted panel sequencing in pediatric primary cardiomyopathy supports a critical role of TNNI3. Clin Genet, 96, 549–559.
https://doi.org/10.1111/cge.13645

Marklewitz M., Dutari L.C., Paraskevopoulou S., Page R.A., Loaiza J.R., Junglen S. (2019). 
"Diverse novel phleboviruses in sandflies from the Panama Canal area, Central Panama."
*Journal of General Virology*
2019 May 3.
[doi: 10.1099/jgv.0.001260](https://doi.org/10.1099/jgv.0.001260)

Quade,A., Thiel,A., Kurth,I., Holtgrewe,M., Elbracht,M., Beule,D., Eggermann,K., Scholl,U.I. and Häusler,M. (2019) Paroxysmal tonic upgaze: A heterogeneous clinical condition responsive to carbonic anhydrase inhibition. European Journal of Paediatric Neurology, [10.1016/j.ejpn.2019.11.002](https://doi.org/10.1016/j.ejpn.2019.11.002).

### 2018

Blanc, E., Holtgrewe, M., Dhamodaran, A., Messerschmidt, C., Willimsky, G., Blankenstein, T., Beule, D. (2018).
"Identification and Ranking of Recurrent Neo-Epitopes in Cancer".
*bioRxiv*.
2018/389437, 2018.
[doi: 10.1101/389437](https://doi.org/10.1101/389437)

Brandt, R., Uhlitz, F., Riemer, P., Giesecke, C., Schulze, S., El-Shimy, I.A., Fauler, B., Mielke, T., Mages, N., Herrmann, B.G., Sers, C., Blüthgen, N., Morkel, M. (2018).
"Cell type-dependent differential activation of ERK by oncogenic KRAS or BRAF in the mouse intestinal epithelium".
*bioRxiv*.
2018/340844.
[doi: 10.1101/340844](https://dx.doi.org/10.1101/340844).

Holtgrewe, M., Knaus, A., Hildebrand, G., Pantel, J.-T., Rodriguesz de los Santos, M., Neveling, K., Goldmann, J., Schubach, M., Jäger, M., Couterier, M., Mundlos, S., Beule, D., Sperling, K., Krawitz, P. (2018).
"Multisite de novo mutations in human offspring after paternal exposure to ionizing radiation",
*Nature Scientific Reports*. 2018 Oct 2;8(1):14611.
[doi: 10.1038/s41598-018-33066-x](https://doi.org/10.1038/s41598-018-33066-x).

Kircher M., Xiong C., Martin B, Schubach M, Inoue F, Bell R.JA., Costello J.F., Shendure J., Ahituv N. (2018).
"Saturation mutagenesis of disease-associated regulatory elements." bioRxiv (2018): 505362. [doi: 10.1101/505362](https://doi.org/10.1101/505362)

PCAWG Transcriptome Core Group, Calabrese, C., Davidson, N.R., Fonseca1, N.A., He, Y., Kahles, A., Lehmann, K.-V., Liu, F., Shiraishi, Y., Soulette, C.M., Urban, L., Demircioğlu, D., Greger, L., Li, S., Liu, D., Perry, M.D., Xiang, L., Zhang, F., Zhang, J., Bailey, P., Erkek, S., Hoadley, K.A., Hou, Y., Kilpinen, H., Korbel, J.O., Marin, M.G., Markowski, J., Nandi11, T., Pan-Hammarström, Q., Pedamallu, C.S., Siebert, R., Stark, S.G., Su, H., Tan, P., Waszak, S.M., Yung, C., Zhu, S., PCAWG Transcriptome Working Group, Awadalla, P., Creighton, C.J., Meyerson, M., Ouellette, B.F.F., Wu, K., Yang, H., ICGC/TCGA Pan-Cancer Analysis of Whole Genomes Network, Brazma1, A., Brooks, A.N., Göke, J., Rätsch, G., Schwarz, R.F., Stegle, O., Zhang, Z. (2018).
"Genomic basis for RNA alterations revealed by whole-genome analyses of 27 cancer types".
*bioRxiv*.
2018/183889.
[doi: 10.1101/183889 ](https://doi.org/10.1101/183889)

Guneykaya D., Ivanov A., Hernandez D.P., Haage V., Wojtas B., Meyer N., Maricos M., Jordan P., Buonfiglioli A., Gielniewski B., Ochocka N., Cömert, C., Friedrich, C., Artiles, L. S., Kaminska, B., Mertins, P., Beule, D., Kettenmann, H. (2018).
"Transcriptional and translational differences of microglia from male and female brains",
*Cell reports*. 2018 Sep 4;24(10):2773-83.
[doi: 10.1016/j.celrep.2018.08.001](https://doi.org/10.1016/j.celrep.2018.08.001).

Rentzsch P, Witten D, Cooper GM, Shendure J, Kircher M. (2018).
"CADD: predicting the deleteriousness of variants throughout the human genome",
*Nucleic Acids Res*. 2018 Oct 29. [doi: 10.1093/nar/gky1016](https://doi.org/10.1093/nar/gky1016).

Salatzki J., Foryst-Ludwig A., Bentele K., Blumrich A., Smeir E., Ban Z., Brix S., Grune J., Beyhoff N., Klopfleisch R., Dunst S., Surma, M.A., Klose, C., Rothe, M., Heinzel, F.R., Krannich, A., Kershaw, E.E., Beule, D., Schulze, P.C., Marx, N., Kintscher, U. (2018).
"Adipose tissue ATGL modifies the cardiac lipidome in pressure-overload-induced left ventricular failure",
*PLoS genetics*. 2018 Jan 10;14(1):e1007171.
[doi: 10.1371/journal.pgen.100717](https://doi.org/10.1371/journal.pgen.1007171).

Schubach M., Re M., Robinson P.N., Valentini G. (2017)
"Imbalance-aware machine learning for predicting rare and common disease-associated non-coding variants",
*Scientific reports* 
7:1, 2959.
[doi: 10.1038/s41598-017-03011-5](https://doi.org/10.1038/s41598-017-03011-5).

Schubert M., Klinge, B., Klünemann M., Sieber A., Uhlitz F., Sauer S., Garnett M., Blüthgen N., Saez-Rodriguez J. (2018).
"Perturbation-response genes reveal signaling footprints in cancer gene expression". 
*Nature Communications*.
9: 20, 2018.
[doi: 10.1038/s41467-017-02391-6](https://doi.org/10.1038/s41467-017-02391-6)

### 2017

Euskirchen, P., Bielle, F., Labreche, K., Kloosterman, W.P., Rosenberg, S., Daniau, M., Schmitt, C., Masliah-Planchon, J., Bourdeaut, F., Dehais, C., et al. (2017). Same-day genomic and epigenomic diagnosis of brain tumors using real-time nanopore sequencing. Acta Neuropathol 1–13. [doi: 10.1007/s00401-017-1743-5](https://doi.org/10.1007/s00401-017-1743-5)

Euskirchen, P., Radke, J., Schmidt, M.S., Heuling, E.S., Kadikowski, E., Maricos, M., Knab, F., Grittner, U., Zerbe, N., Czabanka, M., et al. (2017). Cellular heterogeneity contributes to subtype-specific expression of ZEB1 in human glioblastoma. PLOS ONE 12, e0185376. [doi: 10.1371/journal.pone.0185376](https://doi.org/10.1371/journal.pone.0185376 )

Mattei D., Ivanov A., Ferrai C., Jordan P., Guneykaya D., Buonfiglioli A., Schaafsma W., Przanowski P., Deuther-Conrad W., Brust P., Hesse S., Patt, M., Sabri, O., Ross, T.L., Eggen, B.J.L., Boddeke E.W.G.M., Kaminska, B., Beule, D., Pombo, A., Kettenmann, H., Wolf, S.A. (2017).
"Maternal immune activation results in complex microglial transcriptome signature in the adult offspring that is reversed by minocycline treatment." *Translational psychiatry*. 2017 May;7(5):e1120.
[doi: 10.1038/tp.2017.80](https://doi.org/10.1038/tp.2017.80).

Mamlouk, S., Childs, L. H., Aust, D., Heim, D., Melching, F., Oliveira, C., Wolf, T., Durek, P., Schumacher, D., Bläker, H., von Winterfeld, M., Gastl, B., Möhr, K., Menne, A., Zeugner, S., Redmer, T., Lenze, D., Tierling, S., Möbs, M., Weichert, W., Folprecht, G., Blanc, E., Beule, D., Schäfer, R., Morkel, M., Klauschen, F., Leser, U. and Sers, C. (2017).
"DNA copy number changes define spatial patterns of heterogeneity in colorectal cancer",
*Nature Communications*. 2017; 8, p. 14093.
[doi: 10.1038/ncomms14093](http://doi.org/10.1038/ncomms14093).

Messerschmidt, C., Holtgrewe, M. and Beule, D. (2017).
"HLA-MA: simple yet powerful matching of samples using HLA typing results".
*Bioinformatics*. 28, pp. 2592–2599.
[doi: 10.1093/bioinformatics/btx132](https://doi.org/10.1101/066548).

Kammertoens, T., Friese, C., Arina, A., Idel, C., Briesemeister, D., Rothe, M., Ivanov, A., Szymborska, A., Patone, G., Kunz, S., Sommermeyer, D., Engels, B., Leisegang, M., Textor, A., Fehling, H. J., Fruttiger, M., Lohoff, M., Herrmann, A., Yu, H., Weichselbaum, R., Uckert, W., Hübner, N., Gerhardt, H., Beule, D., Schreiber, H. and Blankenstein, T. (2017).
"Tumour ischaemia by interferon-γ resembles physiological blood vessel regression".
*Nature*. 545(7652), pp. 98–102.
[doi: 10.1038/nature22311](https://doi.org/10.1038/nature22311).

Schulze Heuling, E., Knab, F., Radke, J., Eskilsson, E., Martinez-Ledesma, E., Koch, A., Czabanka, M., Dieterich, C., Verhaak, R.G., Harms, C., et al. (2017). Prognostic Relevance of Tumor Purity and Interaction with MGMT Methylation in Glioblastoma. Mol. Cancer Res. 15, 532–540. [doi: 10.1158/1541-7786.MCR-16-0322 ](https://doi.org/10.1158/1541-7786.MCR-16-0322 )

Yaakov, G., Lerner, D., Bentele, K., Steinberger, J., Barkai, N., Bigger, J., Maisonneuve, E., Gerdes, K., Lewis, K., Dhar, N., McKinney, J. D., Gefen, O., Balaban, N. Q., Jayaraman, R., Balaban, N. Q., Merrin, J., Chait, R., Kowalik, L., Leibler, S., Balaban, N. Q., Allison, K. R., Brynildsen, M. P., Collins, J. J., Nathan, C., Lewis, K., Glickman, M. S., Sawyers, Knoechel, B., Welch, A. Z., Gibney, P. A., Botstein, D., Koshland, D. E., Levy, S. F., Ziv, N., Siegal, M. L., Stewart-Ornstein, J., Weissman, J. S., El-Samad, H., Gasch, A. P., Weinert, T., Hartwell, L., Weinert, T. A., Hartwell, L. H., Lisby, M., Rothstein, R., Mortensen, U. H., Lisby, M., Mortensen, U. H., Rothstein, R., Domkin, V., Thelander, L., Chabes, A., Hendry, J. A., Tan, G., Ou, J., Boone, C., Brown, G. W., Berry, D. B., Gasch, A. P., Lynch, M., Nishant, K. T., Serero, A., Jubin, C., Loeillet, S., Legoix-Ne, P., Nicolas, A. G., Huh, W. K., Janke, C., Lee, S. E., Blecher-Gonen, R., Martin, M., Cherry, J. M., McKenna, A., DePristo, M. A., Lawrence, M., Obenchain, V., Ye, K., Schulz, M. H., Long, Q., Apweiler, R., Ning, Z., Layer, R. M., Chiang, C., Quinlan, A. R., Hall, I. M., Faust, G. G., Hall, I. M., Boeva, V., Boeva, V., Li, H., Koren, A., Soifer, I. and Barkai, N. (2017).
"Coupling phenotypic persistence to DNA damage increases genetic diversity in severe stress".
*Nature Ecology & Evolution*. 1(1), pp. 497–500.
[doi: 10.1038/s41559-016-0016](https://doi.org/10.1038/s41559-016-0016).

Uhlitz, F., Sieber, A., Wyler, E., Fritsche-Guenther, R., Meisig, J., Landthaler, M., Klinger, B., Blüthgen, N. (2017).
"An immediate-late gene expression module decodes ERK signal duration".
*Molecular Systems Biology*.
13: 928, 2017.
[doi: 10.15252/msb.20177554](https://doi.org/10.15252/msb.20177554).

## Theses

### 2019

Schumann F. (2019).
"Establishing a pipeline for stable mutational signature detection and evaluation of variant filter effects". *Freie Universität Berlin*.
Bachelor Thesis, Bioinformatics.

### 2018

Borgsmüller N. (2018).
"Optimization of data processing in GC-MS metabolomics",
*Technische Universität Berlin*.
Master Thesis, Biotechnology.

Kuchenbecker, S.-L. (2018).
"Analysis of Antigen Receptor Repertoires Captured by High Throughput Sequencing".
*Freie Universität Universität Berlin*.
PhD Thesis, Dr. rer. nat.
[URN:NBN: urn:nbn:de:kobv:188-refubium-22171-8](https://nbn-resolving.de/urn:nbn:de:kobv:188-refubium-22171-8)

Schubach M. (2018).
"Learning the Non-Coding Genome",
*Freie Universität Universität Berlin*.
PhD Thesis, Dr. rer. nat.
[URN:NBN: urn:nbn:de:kobv:188-refubium-23332-7](https://nbn-resolving.de/urn:nbn:de:kobv:188-refubium-23332-7)

## Posters

### 2018

Roskosch, S., Haldórsson B., Kehr, B. (2018).
"PopDel: Population-Scale Detection of Genomic Deletions"
*ECCB 2018.*
Poster.

White T., Kehr B. (2018).
"Comprehensive extraction of structural variations from long-read DNA sequences"
*WABI 2018.*
Poster.

### 2017

Schubach M., Re R., Robinson P.N., Valentini G. (2017).
"Variant relevance prediction in extremely imbalanced training sets"
*ISMB/ECCB 2017.*
Poster.

White T., Kehr B. (2017).
"Improving long-read mapping with simple lossy sequence transforms"
*ISMB/ECCB 2017.*
Poster.
